{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "8e64154f",
   "metadata": {},
   "source": [
    "### Webscraper for extracting newly published names, strains and accession numbers from the weeekly IJSEM email (saved in html). Script then compares the IJSEM names to the NCBI names and generates a report used for taxonomy updates. \n",
    "### Uses REGEX to extract information from text\n",
    "### Notes on selenium. There is a lot of bad advise in stack overflow that refers to old versions of selenium. Need to refer to the offocial documentation https://selenium-python.readthedocs.io/ If you run out of space in your linux directory the script will crash. There is a cache file written by selenium that may then need to be deleted to get the chrome driver working again. Follow the path in the error message and delete that cache directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4657a12a",
   "metadata": {},
   "outputs": [],
   "source": [
    "pip install \"numpy<2.0\" "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a58e07e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "import selenium\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.common.by import By\n",
    "#PATH = \"C:\\Users\\mcveigh\\Documents\\PythonPC\\chrome.exe\"\n",
    "#driver = webdriver.Chrome(PATH)\n",
    "from selenium.webdriver.chrome.options import Options\n",
    "import time\n",
    "import pandas as pd\n",
    "import re\n",
    "import os\n",
    "import sys\n",
    "import bs4\n",
    "from bs4 import BeautifulSoup\n",
    "import requests\n",
    "import numpy as np\n",
    "from datetime import datetime\n",
    "from nameparser import HumanName"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0ce4b6ac",
   "metadata": {},
   "source": [
    "### List of URLs input to search short list manually entered"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3fbfe8be",
   "metadata": {},
   "outputs": [],
   "source": [
    "urls = ['https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006523?emailalert=true',\n",
    "       'https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006528?emailalert=true',\n",
    "        'https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006510?emailalert=true',\n",
    "        'https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006453?emailalert=true'\n",
    "       ]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4a6a4c13",
   "metadata": {},
   "source": [
    "### Find URLS from saved email in html - save as from outlook in htm format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "6136fcf9",
   "metadata": {},
   "outputs": [],
   "source": [
    "input = (r'IJSEMemail29.htm')\n",
    "output = (r'NameCheckweek29a.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "58762373",
   "metadata": {},
   "outputs": [],
   "source": [
    "#with open ('IJSEMemail1.htm', encoding = 'unicode_escape') as f:\n",
    "with open (input, encoding = 'unicode_escape') as f:\n",
    "    content = f.read()\n",
    "    soup = BeautifulSoup(content, 'html.parser')  \n",
    "#print(soup.prettify())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "eae08e3b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<a alt=\"Discover publishing discounts and more by becoming a member\" href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrobiologysociety.org%2Fmembership%2Fjoin-the-society.html%3Futm_source%3DETOC%26utm_medium%3DBanner%26utm_campaign%3DMembership&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018527510%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=9cAiYToxxTYjt8w%2FgemfeKqTjUbkLDkU5fso9q5TjSE%3D&amp;reserved=0\"><span style=\"text-decoration:none;text-underline:none\"><img border=\"0\" class=\"fr-fic fr-dii\" id=\"_x0000_i1026\" src=\"https://www.microbiologyresearch.org/upload/marketing/ETOC-MemberBenefits-Banner-resized.png\" style=\"max-width:100%\" title=\"Member benefits\"/></span></a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006617%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018546452%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=qPgbRuHApRO85zY1%2FXyHK7fwpRhZIQDtoO%2BMyTwMQ8I%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Roseomonas\n",
      "cutis</span></span> sp. nov., isolated from human epidermis </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006617&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018559798%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=DymnhPHo7jKVkdcTUnv%2FNhrmWcVbJ8at1NRXYGC4ssA%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006617</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006638%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018572696%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=g%2BuTT%2F2f5PKm0UEUOlkdDwx6NxxHxYJMPcdX%2BhfDr3Q%3D&amp;reserved=0\">\n",
      "Addition of Section 10, Rules 6673 for further integration of <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Candidatus</span></span>\n",
      "names into the International Code of Nomenclature of Prokaryotes </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006638&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018585313%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=N%2FyQtLS0SPyKbNcvcR9vKnetN1taEQsD5fN64YAZQIg%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006638</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006629%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018597448%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=gTvoT9taoY5EywTBMznivistMD3RQuxyiE0TB92Vbu8%3D&amp;reserved=0\">\n",
      "Identification of <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Starmerella\n",
      "aleppica</span></span> f.a., sp. nov. and large indels in the rRNA cistron that\n",
      "split the <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Starmerella</span></span>\n",
      "genus </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006629&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018609688%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=STm2wx%2BTl7zewkX6odA4oEZn1THm6C3uHx0R4smPQAI%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006629</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006625%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018622625%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=rKq32rkBWGoYhHMlOxdz%2B1l3hOh1haV9AJluizjDWsg%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudoalteromonas\n",
      "qingdaonensis</span></span> sp. nov., isolated from the intestines of <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Ilyoplax\n",
      "deschampsi</span></span> </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006625&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018635147%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2BHtdn%2BPfWd6EKHtxsE1JLQhnhg%2B%2FEzxFxoggSNZbM%2Fk%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006625</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Methylobacterium\n",
      "litchii</span></span> sp. nov., <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>Methylobacterium guangdongense</span></span> sp. nov. and <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Methylorubrum\n",
      "subtropicum</span></span> sp. nov.<span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>,</span></span> isolated from the phyllosphere of litchi </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006639&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018662612%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=hnyYSwYvcLbeCTMT2gLfflxMvVsT%2FMy2KbQabxPH%2B6U%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006639</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudomonas\n",
      "fungipugnans</span></span> sp. nov., a potently antifungal bacterium isolated\n",
      "from moss </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006624&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018687247%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=zp6a%2FJFhRjAikfmebL4jLkr%2BBwiIe%2B0PuKQEl4glLLw%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006624</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006628%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018699661%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=ESNDly52%2BOjW9hU5mGCLuXjU%2BggyJLmc5bboYWHosp0%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Paenibacillus\n",
      "zeirhizosphaerae</span></span> sp. nov., isolated from surface of the maize (<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Zea mays</span></span>)\n",
      "roots in a horticulture field, Hungary </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006628&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018712204%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=y07%2F0yMKTPnRWhgw%2BrKP%2FgK3iW0Zh97gydQ7G78nXP4%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006628</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Leptospira\n",
      "gorisiae</span></span> sp. nov, <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>L</span></span>. <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>cinconiae</span></span> sp. nov, <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>L</span></span>. <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>mgodei</span></span>\n",
      "sp. nov, <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>L</span></span>.\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>milleri</span></span>\n",
      "sp. nov and <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>L</span></span>.\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>iowaensis</span></span>\n",
      "sp. nov: five new species isolated from water sources in the Midwestern United\n",
      "States </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006595&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018737672%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=MSRmjb41GXGulTGpF78fqGOC0dXIstigOF%2FHLbT4CWI%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006595</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006620%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018750576%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=4wwgiVC%2BIgOfH2hx4iF0tX5DG7sH4sk5ExX5yA9s9Eo%3D&amp;reserved=0\">\n",
      "Four novel nontuberculous mycobacteria species: <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Mycobacterium wendilense</span></span>\n",
      "sp. nov., <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Mycobacterium\n",
      "burgundiense</span></span> sp. nov., <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Mycobacterium kokjensenii</span></span>\n",
      "sp. nov. and <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Mycobacterium\n",
      "holstebronense</span></span> sp. nov. revived from a historical Danish strain\n",
      "collection </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006620&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018763426%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=TG%2Flr0FVcx7X6XgAnV2iCMN%2BmYGvn5IzWSILQecJhSk%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006620</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006627%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018776116%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=N%2B6rCRwRGR2hPXGMDf2HI73r1Z9dNMUsDxlZPvhqbHQ%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Tistrella\n",
      "arctica</span></span> sp. nov., isolated from the surface seawater of the\n",
      "Bering Sea and Chukchi Sea and emended description of the genus <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Tistrella</span></span>\n",
      "</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006627&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018788861%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2FNiViIMa4HSzJGJCuyYCZoez0x2hDsXXK8QVAORyLx0%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006627</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Luteimonas\n",
      "salinilitoris</span></span> sp. nov., isolated from the shore soil of saline\n",
      "lake in Tibet of China </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006630&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018815219%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=9E2zQpZK7oPm2WnQU5%2FeXKNRvk8YMU8FpL6gUOqrf00%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006630</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006605%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018827923%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Dn9l%2FcNamFlaRr6m09h5gWnKm5mN98SKOE89dm0PQas%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Tenacibaculum\n",
      "platacis</span></span> sp. nov., <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>Tenacibaculum vairaonense</span></span> sp. nov. and <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Tenacibaculum\n",
      "polynesiense</span></span> sp. nov. isolated from batfish (<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Platax orbicularis</span></span>)\n",
      "in Tahiti Island, French Polynesia </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006605&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018840574%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=c%2FTbkgPUPdGVt7npvQ9wlvaO4TLBxLkdwj5SHZfgeSM%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006605</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006611%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018853339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=S4R1T9Bs6FJngYjedz8T7GqhIOrzYpyrq3%2BkVjxy5%2Bw%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Dyella\n",
      "aluminiiresistens</span></span> sp. nov., a Al<span class=\"jp-sup\"><span style='font-family:\"Arial\",sans-serif'>3+</span></span>-tolerant bacterium with\n",
      "the ability to inhibit <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Fusarium\n",
      "oxysporum</span></span> f. sp. <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>melonis</span></span> isolated from rhizosphere soil of\n",
      "muskmelon </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006611&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018865735%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=a0KVhEyonrzpRIR1LtB%2Fy%2Fsm1JbB4XQYkEw92VR5H1Y%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006611</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006626%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018878014%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=yxI7HBcs%2Bj2A87QDGqjqlFq02CSGcrFK%2B4TNDys38f0%3D&amp;reserved=0\">\n",
      "Emendation of Recommendation 6(7), Rule 64 and Appendix 9 Section D of the\n",
      "International Code of Nomenclature of Prokaryotes to regulate the formation of\n",
      "prokaryote names from personal names </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006626&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018890067%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=0%2FVC6738J2HA43Vj7itbD2s0IU2mYrgzzoWYVVxYVBc%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006626</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006622%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018902382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=wFG5W6%2FaDt6seMBNjmLMyDDd%2Be3yRt5juFlPJLI%2BzS4%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Chionosphaera\n",
      "pinicorticola</span></span> sp. nov., a novel basidiomycetous yeast species\n",
      "isolated from pine tree bark in Gyeongju, South Korea </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006622&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018914953%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=fSqt%2FhDSHVkRd5mNfTCc6bDlku%2BSDUMPylem%2Fu3mRhE%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006622</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006621%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018927362%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=QjfUqhPoNrrsl%2Bp3CIzDHpQTkgGBFAofKtHEzxhDn38%3D&amp;reserved=0\">\n",
      "International Committee on Systematics of Prokaryotes: minutes of the open\n",
      "plenary meeting, Tuesday, 22 October 2024, Sesto Fiorentino, Florence, Italy,\n",
      "and via Teams </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006621&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018939953%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=mi9Amp5rg%2BlNfmpavaKU0iW0kSH%2FSvbKBhfvFUw9YXs%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006621</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudonocardia\n",
      "spirodelae</span></span> sp. nov., isolated from duckweed and formal proposal\n",
      "to reclassify <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudonocardia\n",
      "antarctica</span></span> as a later heterotypic synonym of <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudonocardia\n",
      "alni</span></span> and reclassify <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>Pseudonocardia carboxydivorans</span></span> as <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Pseudonocardia\n",
      "alni</span></span> subsp. <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>carboxydivorans</span></span>\n",
      "</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006608&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018966103%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=gVspNhWtqH%2FNurDy9MDfGemt6onxlvx5uiPoCmwgp0A%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006608</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006616%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018979116%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=xd6O7fsm7ShyVHqhXbw4VOdozu6kXctjKveMb16GMgw%3D&amp;reserved=0\">\n",
      "Genome-based reclassification of <span class=\"jp-italic\"><span style='font-family:\n",
      "\"Arial\",sans-serif'>Suonthocola fibrivorans</span></span> as a later\n",
      "heterotypic synonym of <span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Diplocloster\n",
      "agilis</span></span> </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006616&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018991468%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=vGp4KCigEByCh8OpHX9QyZnR28koUXy0msmWf6d3nqw%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006616</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&amp;reserved=0\">\n",
      "<span class=\"jp-italic\"><span style='font-family:\"Arial\",sans-serif'>Vishniacozyma\n",
      "siamensis</span></span> sp. nov., a new anamorphic tremellomycetous yeast\n",
      "species isolated from a mangrove forest in Thailand </a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fdx.doi.org%2F10.1099%2Fijsem.0.006623&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019016457%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=HUXKWgY%2FZp4Gh9Pe%2BqInuZsfspGKdWF2mD75u3146Y4%3D&amp;reserved=0\" target=\"_blank\">http://dx.doi.org/10.1099/ijsem.0.006623</a>\n",
      "<a href=\"https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fmyaccount&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019028813%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Eh7G2vGpQAfRPqH3oLioAy6zBxjxmtExma79R4Jk3UM%3D&amp;reserved=0\">https://www.microbiologyresearch.org/myaccount</a>\n",
      "40\n"
     ]
    }
   ],
   "source": [
    "urls = []\n",
    "numberoflinks = 0\n",
    "for url in soup.findAll( 'a', attrs={'href': re.compile(\"^https:\")}):\n",
    "    urls.append(url.get('href'))\n",
    "    numberoflinks = numberoflinks + 1\n",
    "    print(url)\n",
    "print(numberoflinks)\n",
    "#print(urls)\n",
    "#remove_list = ['TandC','doi.org', 'myaccount']\n",
    "urls = [e for e in urls if \"TandC\" not in e and \"doi.org\" not in e and \"myaccount\" not in e and \"join-the-society\" not in e]\n",
    "\n",
    "#print(dois)\n",
    "#print(urls)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f7e20bf4",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "33443ded",
   "metadata": {},
   "source": [
    "### Main body - Selenium to extract data from html\n",
    "### https://nameparser.readthedocs.io/en/latest/index.html to find the last name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "22d6f4bf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Roseomonas cutis sp. nov., isolated from human epidermis\n",
      "Description of Roseomonas cutis sp. nov.\n",
      " Roseomonas cutis (cutis. L. gen. n. cutis, of the skin)\n",
      "Cells are Gram-stain-negative, pink pigmented, strictly aerobic, motile and rod shaped. Cell sizes are 1.41.60.60.7 m. Can assimilate sodium pyruvate, d-fructose, sucrose, d-mannose, disodium succinate, dl-xylose, dl-arabinose, N-acetylglucosamine, maltose, sodium gluconate, sodium maleate and d-galacturonic acid. But, cannot assimilate d-glucose, glycerol, lactose, d-sorbitol, d-mannitol, d-galactose, d-rhamnose, sodium acetate and trisodium citrate. Oxidase and catalase are positive. Growth occurs at 740 C (optimum, 28 C) and pH 6.09.5 (optimum, pH 7.0), and cells tolerate up to 3% NaCl; positive for urease, starch hydrolysis, carboxymethyl cellulose hydrolysis and DNase activities; negative for haemolysis, methyl red test, coagulase, gelatin, Tween 20, Tween 80 and casein hydrolysis activities. Nitrate is reduced to nitrite. In API ZYM, cells are positive for alkaline phosphatase, esterase (C4), esterase lipase (C8), valine arylamidase, acid phosphatase and naphthol-AS-BI-phosphohydrolase and negative for lipase (C14), leucine arylamidase, crystine arylamidase, trypsin, -chymotrypsin, - and -galactosidase, -glucuronidase, - and -glucosidase, N-acetyl--glucosaminidase, -mannosidase and -fucosidase activities. Major cellular fatty acids are summed feature 8 (C18:1 7c and/or C18:1 6c) and C18:1 2OH, C16:0, C16:1 5c and summed feature 3 (C16:1 7c and/or C16:1 6c). Only Q-10 is detected as the respiratory quinone. Phosphatidylethanolamine, phosphatidylcholine, phosphatidylglycerol, diphosphatidylglycerol and two aminolipids are detected as major polar lipids.\n",
      "The type strain is OT10T (=KCTC 92087T =JCM 34968T), isolated from male skin swab sample in South Korea. The DNA G+C content of the type strain is 72.6 mol% (genome). The GenBank accession numbers of the 16S rRNA gene and whole-genome sequences of strain OT10T are OP788068 and ASM2099108v1, respectively.\n",
      " \n",
      "orgname ['Roseomonas cutis ']\n",
      "accessions ['OP788068']\n",
      "strain names [' is OT10T (=KCTC 92087T =JCM 34968T', ' is 72.6 mol% (genome']\n",
      "Description of Roseomonas cutis sp. nov.\n",
      "orgname ['Roseomonas cutis ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Addition of Section 10, Rules 66–73 for further integration of Candidatus names into the International Code of Nomenclature of Prokaryotes\n",
      "Identification of Starmerella aleppica f.a., sp. nov. and large indels in the rRNA cistron that split the Starmerella genus\n",
      "Description of Starmerella aleppica f.a. Sipiczki and Baghela sp. nov\n",
      " Starmerella aleppica (a.leppi.ca. N.L. fem. adj. aleppica, referring to Aleppo, where the flowers were collected for yeast isolation).\n",
      "In colonies formed on yeast-extract-glucose agar plates (YEA) and in yeast-extract-glucose broth (YEL) after 1 day of incubation at 25 C, cells are ovoid (46 x 35 m), occurring (mainly) singly or in pairs. Vegetative reproduction proceeds by apical or subapical budding. On YEA after 3 weeks of incubation at 25 C, colonies are slightly pink, butyrous and soft with the entire margin. In cultures inoculated on thin films of YEA sandwiched between glass slides and cover slips, neither pseudohyphae nor true hyphae are produced. On synthetic minimal medium (EMMA: Edinburgh minimal medium agar), invasive rudimentary pseudohyphae consisting of loosely connected, slightly elongated cells are visible under the colonies. Asci or signs of conjugation were not seen in any strains or their mixtures on YEA, malt-extract agar, corn-meal agar, acetate agar and Gorodkowa agar. Glucose and sucrose are fermented. Assimilation of carbon compounds: glucose, sucrose, sorbose, raffinose, maltose (weak), inulin (weak), glycerol (weak), gluconate, methanol (weak) and ethanol (weak). No growth occurs on galactose, d-glucosamine, ribose, xylose, l-arabinose, d-arabinose, rhamnose, trehalose, methyl--d-glucoside, cellobiose, salicin, arbutin, melibiose, lactose, melesitose, starch, meso-erythritol, ribitol, xylitol, l-arabinitol, d-mannitol, galactitol, myo-inositol, d-glucuronate, dl-lactate, succinate, citrate, butane 2,3 diol and quinic acid. Assimilation of nitrogen compounds: positive for lysine, creatinine, glucoamine, d-tryptophan and negative for nitrate, nitrite, ethylamine, cadaverine, imidazole and quinic acid. Acid production is negative. Starch-like compounds are not produced. Growth with 0.01% cycloheximide is negative. Growth on 50% and 60% glucose is positive. No growth occurs at 37 C. Growth in vitamin-free medium is positive. The holotype CBS 12960 (original strain number 2-1361) was isolated from a Malva sylvestris flower in Syria in 2009 and is preserved in a metabolically inactive state in the CBS Yeast Collection of the Westerdijk Fungal Biodiversity Institute, Utrecht, the Netherlands. Ex-type cultures have been deposited in the Culture Collection of Yeasts, Bratislava, Slovakia (CCY 90-2-1) and in the National Collection of Agricultural and Industrial Microorganisms, Budapest, Hungary (NCAIM Y.02123). MycoBank: MB855459.\n",
      " \n",
      "orgname ['and Baghela ']\n",
      "accessions ['MB855459']\n",
      "strain names []\n",
      "Description of Starmerella aleppica f.a. Sipiczki and Baghela sp. nov\n",
      "orgname ['and Baghela ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Pseudoalteromonas qingdaonensis sp. nov., isolated from the intestines of Ilyoplax deschampsi\n",
      "Methylobacterium litchii sp. nov., Methylobacterium guangdongense sp. nov. and Methylorubrum subtropicum sp. nov., isolated from the phyllosphere of litchi\n",
      "Description of Methylobacterium litchii sp. nov.\n",
      " Methylobacterium litchii (litchi.i. N.L. gen. masc. n. litchii, of the fruit of Litchi chinensis, referring to what the species was isolated from).\n",
      "Cells are Gram-stain-negative, aerobic, rod-shaped and motile. Able to grow on R2A, TSA and NA. Colonies are pink-pigmented, untransparent, circular, convex, smooth and 0.50.7 mm in diameter after 5 days incubation on R2A agar at 28 C. Growth occurs at 1037 C (optimum, 2535 C) and pH 5.08.0 (optimum, pH 7.0) and in the presence of 02% NaCl (w/v) (optimum without NaCl). It can utilize methanol as the sole carbon source, but not methylamine or formaldehyde. Positive for oxidase, catalase, urease, siderophore production, hydrolyses of Tween 40 and aesculin, assimilation of glucose, gluconate and citrate, but negative for nitrate reduction, indole production, fermentation of glucose, arginine dihydrolase, -galactosidase, hydrolyses of gelatin, starch and Tween 80, assimilation of arabinose, mannose, mannitol, N-acetyl-glucosamine, maltose, capric acid, adipic acid, malate and phenylacetic acid. The predominant respiratory quinone is ubiquinone 10 and the major fatty acid is summed feature 8 (C18:1 7c and/or C18:1 6c). Polar lipids contain phosphatidylglycerol, diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylcholine, an unidentified aminolipid and an unidentified lipid.\n",
      "The type strain is SD21T (=GDMCC 1.4327T=KCTC 8300T), which was isolated from the phyllosphere of litchi in Xujiang Town located in Huizhou, Guangdong Province, PR China. The genomic DNA G+C content of the type strain is 69.0%. The GenBank accession numbers for the 16S rRNA gene and whole-genome sequences are PP600109 and JBBXMY000000000, respectively.\n",
      " \n",
      "orgname ['Methylobacterium litchii ']\n",
      "accessions ['PP600109', 'JBBXMY000000000']\n",
      "strain names [' is SD21T (=GDMCC 1.4327T=KCTC 8300T']\n",
      "Description of Methylobacterium litchii sp. nov.\n",
      "orgname ['Methylobacterium litchii ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Methylobacterium guangdongense sp. nov.\n",
      " Methylobacterium guangdongense (guang.dong.ense. N.L. neut. adj. guangdongense, originating from Guangdong Province of PR China, from where the strains were isolated).\n",
      "Cells are Gram-stain-negative, aerobic, rod-shaped and motile. Able to grow on R2A, TSA and NA. Colonies are pink-pigmented, untransparent, circular, convex, smooth, and 0.50.7 mm in diameter after 5 days incubation on R2A agar at 28 C. Growth occurs at 1037 C (optimum, 2535 C) and pH 5.08.0 (optimum, 7.0) and in the presence of 00.5% NaCl (w/v) (optimum without NaCl). It can utilize methanol as the sole carbon source, but not methylamine or formaldehyde. Positive for oxidase, catalase, urease, hydrolysis of Tween 40 and aesculin, assimilation of glucose, arabinose, mannitol, maltose, gluconate and adipic acid, but negative for nitrate reduction, indole production, siderophore production, fermentation of glucose, arginine dihydrolase, -galactosidase, hydrolyses of gelatin, starch and Tween 80 and assimilation of mannose, N-acetyl-glucosamine, capric acid, citrate, malate and phenylacetic acid. The predominant respiratory quinone is ubiquinone 10 (Q-10) and the major fatty acid is summed feature 8 (C18:1 7c and/or C18:1 6c). Polar lipids contain phosphatidylglycerol, diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylcholine, an unidentified aminolipid and an unidentified lipid.\n",
      "The type strain is SI9T (=GDMCC 1.14329T= KCTC 8298T), which was isolated from the phyllosphere of litchi in Zhenlong Town located in Huizhou, Guangdong Province, PR China. The genomic DNA G+C content of the type strain is 69.1%. The GenBank accession numbers for the 16S rRNA gene and whole genome sequences are PP600110 and JBBXMX000000000, respectively.\n",
      " \n",
      "orgname ['Methylobacterium guangdongense ']\n",
      "accessions ['PP600110', 'JBBXMX000000000']\n",
      "strain names [' is SI9T (=GDMCC 1.14329T= KCTC 8298T']\n",
      "Description of Methylobacterium guangdongense sp. nov.\n",
      "orgname ['Methylobacterium guangdongense ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Methylorubrum subtropicum sp. nov.\n",
      " Methylorubrum subtropicum (sub.tro pi. cum. L. prep. sub, under, below, slightly; L. masc. adj. tropicus, tropical; N.L. neut. adj. subtropicum, subtropical, referring to the subtropical region).\n",
      "Cells are Gram-stain-negative, aerobic, rod-shaped and motile with a single polar flagellum. Able to grow on R2A and NA, but not on TSA. Colonies are pink-pigmented, untransparent, circular, convex, smooth, and 0.50.7 mm in diameter after 5 days incubation on R2A agar at 28 C. Growth occurs at 1535 C (optimum, 2530 C) and pH 6.09.0 (optimum, 7.0) and in the presence of 02% NaCl (w/v) (optimum without NaCl). It can utilize methanol and methylamine as the sole carbon sources, but not formaldehyde. Positive for oxidase, catalase, urease, hydrolysis of aesculin, assimilation of glucose and arabinose, but negative for nitrate reduction, indole production, siderophore production, fermentation of glucose, arginine dihydrolase, -galactosidase, hydrolyses of gelatin, starch and Tweens 40 and 80, assimilation of mannose, N-acetyl-glucosamine, mannitol, capric acid, maltose, gluconate, adipic acid, citrate, malate and phenylacetic acid. The predominant respiratory quinone is ubiquinone 10 (Q-10) and the major fatty acid is summed feature 8 (C18:1 7c and/or C18:1 6c). Polar lipids contain phosphatidylglycerol, diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylglycerol and an unidentified glycolipid.\n",
      "The type strain is SB2T (=GDMCC 1.14328T= KCTC 8299T), which was isolated from the phyllosphere of litchi in Nancheng District located in Dongguan, Guangdong Province, PR China. The genomic DNA G+C content of the type strain is 69.6%. The GenBank accession numbers for the 16S rRNA gene and whole genome sequences are PP600111 and JBBXMW000000000, respectively.\n",
      " \n",
      "orgname ['Methylorubrum subtropicum ']\n",
      "accessions ['PP600111', 'JBBXMW000000000']\n",
      "strain names [' is SB2T (=GDMCC 1.14328T= KCTC 8299T']\n",
      "Description of Methylorubrum subtropicum sp. nov.\n",
      "orgname ['Methylorubrum subtropicum ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pseudomonas fungipugnans sp. nov., a potently antifungal bacterium isolated from moss\n",
      "Description of Pseudomonas fungipugnans sp. nov.\n",
      " Pseudomonas fungipugnans (fun.gi.pugnans. L. masc. n. fungus, mushroom; L. pres. part. pugnans, fighting; N.L. part. adj. fungipugnans, fighting fungi).\n",
      "P. fungipugnans cells are Gram-negative, motile rods (mean length of 2.60 m, mean width of 0.67 m) with one to three polar flagella. The organism is aerobic and gives round, beige mucoid colonies on LB agar at 30 C. It also grows on trypticase soy and M9 salt minimal agar. Optimal growth is at 30 C; however, growth also occurs at low temperatures (8 C). Growth is strong in the presence of 03% NaCl and weak at 4% NaCl, with optimal growth at 1% NaCl. Maximal growth occurs at pH 67, and weak growth at pH 8. Colonies show fluorescence on Kings medium B and produce siderophores on CAS medium. P. fungipugnans is oxidase positive. Cells hydrolyse arginine under anaerobic conditions and produce a hypersensitive response in tobacco leaves. Genomic G+C content is 0.599.\n",
      "Biolog GENIII plate results indicate robust utilization of the following substrates: d-trehalose, N-acetyl-d-glucosamine, -d-glucose, d-fructose, d-galactose, d-mannitol, glycerol, l-alanine, l-arginine, d-mannose, inosine, d-sorbitol, d-arabitol, myo-inositol, d-glucose-6-PO4, l-aspartic acid, l-glutamic acid, l-histidine, l-pyroglutamic acid, l-serine, d-galacturonic acid, l-galactonic acid lactone, d-gluconic acid, d-glucuronic acid, mucic acid, quinic acid, d-saccharic acid, methyl pyruvate, l-lactic acid, citric acid, -keto-glutaric acid, l-malic acid, Tween 40, -amino-butyric acid, -hydroxy-D, L butyric acid, propionic acid and acetic acid. It gave weak positive results for d-fucose, d-fructose-6-PO4, d-serine, gelatin, glycyl-l-proline, glucuronamide, d-malic acid, bromo-succinic acid and acetic acid.\n",
      "P. fungipugnans can be differentiated from closely related species by its positive characteristics in the following tests: tobacco HSR (note that P. nunensis also gives a positive result for this test), potent antifungal activity against F. solani and d-glucose-6-PO4 assimilation.\n",
      "The type strain was isolated from a moss sample and is 681, and no other strains of this species were found. P. fungipugnans sp. nov. has been submitted to the DSMZ and LMG/BCCM strain collections and has the following strain designations in these collections, respectively: DSM 115721 and LMG 33039. The GenBank accession number for the 16S rRNA gene sequence of 681 is OQ547267, and the whole-genome sequence project can be found under JARBWL000000000.\n",
      " \n",
      "orgname ['Pseudomonas fungipugnans ', 'P. fungipugnans ']\n",
      "accessions ['OQ547267', 'JARBWL000000000']\n",
      "strain names []\n",
      "Description of Pseudomonas fungipugnans sp. nov.\n",
      "orgname ['Pseudomonas fungipugnans ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Paenibacillus zeirhizosphaerae sp. nov., isolated from surface of the maize (Zea mays) roots in a horticulture field, Hungary\n",
      "Leptospira gorisiae sp. nov, L. cinconiae sp. nov, L. mgodei sp. nov, L. milleri sp. nov and L. iowaensis sp. nov: five new species isolated from water sources in the Midwestern United States\n",
      "Description of Leptospira mgodei sp. nov.\n",
      " Leptospira mgodei (mgode.i. N.L. gen. n. mgodei, named in honour of Georgies Mgode, a Tanzanian leptospirologist who made significant contributions to the study of human and animal leptospirosis) (Subclade S1).\n",
      "Cells are 12.352.91 m long and 0.16 m in diameter, presenting with a wavelength of ~0.47 m; cells are thin and helical-shaped, with hooked and/or spiral ends to which flagella are inserted. Cells are highly motile and grow better in HAN media at 29 C compared to EMJH media at 29 C or HAN media at 37 C. When cells are seeded onto HAN solid media and incubated at 29 and 37 C, colonies can be observed at 10 days only at 29 C . The genomic G+C content of the type strain WS4.C2T (=NVSL-WS4.C2 T=KIT0305T) is 39.00 mol%.\n",
      " \n",
      "orgname ['Leptospira mgodei ']\n",
      "accessions []\n",
      "strain names [' WS4.C2T (=NVSL-WS4.C2 T=KIT0305T']\n",
      "Description of Leptospira mgodei sp. nov.\n",
      "orgname ['Leptospira mgodei ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Leptospira iowaensis sp. nov.\n",
      " Leptospira iowaensis (i.o.wa.ensis. N.L. fem. adj. iowaensis, of Iowa, named after the state from which the isolate was recovered) (Subclade S1).\n",
      "Cells are 14.92.08 m long and 0.17 m in diameter, presenting with a wavelength of ~0.64 m; cells are thin and helical-shaped, with hooked and/or spiral ends to which flagella are inserted. Cells are highly motile and display vigorous growth at 29 C (EMJH and HAN) and at 37 C (HAN media). When cells are seeded onto HAN solid media and incubated at 29 and 37 C, colonies can be observed at 10 days only at 29 C. The genomic G+C content of the type strain WS39.C2T (=NVSL-WS39.C2 T=KIT0306T) is 37.00 mol%.\n",
      " \n",
      "orgname ['Leptospira iowaensis ']\n",
      "accessions []\n",
      "strain names [' WS39.C2T (=NVSL-WS39.C2 T=KIT0306T']\n",
      "Description of Leptospira iowaensis sp. nov.\n",
      "orgname ['Leptospira iowaensis ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Leptospira cinconiae sp. nov.\n",
      " Leptospira cinconiae (cin.coni.ae. N.L. gen. n. cinconiae, named in honour of Marina Cinco, an Italian leptospirologist who made significant contributions to the study of human and animal leptospirosis) (Subclade P2).\n",
      "Cells are 16.151.18 m long and 0.15 m in diameter, presenting with a wavelength of ~0.71 m; cells are thin and helical-shaped, with hooked and/or spiral ends to which flagella are inserted. Cells are highly motile and display vigorous growth at 29 C in both EMJH and HAN media and at 37 C in HAN media. When cells are seeded onto HAN solid media and incubated at 29 and 37 C, colonies can be observed at 10 days. The genomic G+C content of the type strain WS58.C2T (=NVSL-WS58.C1T=KIT0304T) is 41.76 mol%.\n",
      " \n",
      "orgname ['Leptospira cinconiae ']\n",
      "accessions []\n",
      "strain names [' WS58.C2T (=NVSL-WS58.C1T=KIT0304T']\n",
      "Description of Leptospira cinconiae sp. nov.\n",
      "orgname ['Leptospira cinconiae ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Leptospira milleri sp. nov.\n",
      " Leptospira milleri (mille.ri. N.L. gen. n. milleri, named in honour of James N. Miller, an American spirochetologist who made significant contributions to the study of Treponema, Leptospira and Borrelia) (Subclade S1).\n",
      "Cells are 18.51.2 m long and 0.18 m in diameter, presenting with a wavelength of ~0.71 m; cells are thin and helical-shaped, with hooked and/or spiral ends to which flagella are inserted. Cells are highly motile and display vigorous growth at 29 C in EMJH and HAN media and at 37 C in HAN media. When cells are seeded onto HAN solid media and incubated at 29 and 37 C, colonies can be observed at 10 days. The genomic G+C content of the type strain WS60.C2T (=NVSL-WS60.C2 T=KIT0307T) is 38.70 mol%.\n",
      " \n",
      "orgname ['Leptospira milleri ']\n",
      "accessions []\n",
      "strain names [' WS60.C2T (=NVSL-WS60.C2 T=KIT0307T']\n",
      "Description of Leptospira milleri sp. nov.\n",
      "orgname ['Leptospira milleri ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Leptospira gorisiae sp. nov.\n",
      " Leptospira gorisiae (go.risi.ae. N.L. gen. gorisiae, named in honour of Marga Goris, a Dutch leptospirologist who made significant contributions to the study of human and animal leptospirosis) (Subclade P1).\n",
      "Cells are 14.424.34 m long and 0.12 m in diameter, presenting with a wavelength of ~0.55 m; cells are thin and helical-shaped, with hooked and/or spiral ends to which flagella are inserted. Cells are highly motile and display vigorous growth at 29 C in EMJH and HAN media and at 37 C in HAN media. When cells are seeded onto HAN solid media and incubated at 29 and 37 C, colonies can be observed at 10 days only at 29 C. The genomic G+C content of the type strain WS92.C1T (=NVSL-WS92.C1T=KIT0303T) is 41.50 mol%.\n",
      "The GenBank accession numbers for 16S rRNA sequences for L. cinconiae strain WS58.C1T, L. gorisiae strain WS92.C1T, L. iowaensis strain WS39.C2T, L. mgodei strain WS4.C2T and L. milleri strain WS60.C2T are PQ066384, PQ066385, PQ066386, PQ066387 and PQ066388, respectively, and the GenBank accession numbers for their genomes are CP162137-CP162138, CP162130-CP162132, CP162142-CP162144, CP162139-CP162141 and CP162133-CP162136, respectively.\n",
      "16S GenBank Accession Numbers:\n",
      "Toggle display:   \n",
      "Open fullscreen\n",
      "#Accession\n",
      "Sequence ID\n",
      "PQ066384\n",
      "Leptospira_cinconiae_WS58.C1\n",
      "PQ066385\n",
      "Leptospira_gorisiae_WS92.C1\n",
      "PQ066386\n",
      "Leptospira_iowaensis_WS39.C2\n",
      "PQ066387\n",
      "Leptospira_mgodei_WS4.C2\n",
      "PQ066388\n",
      "Leptospira_milleri_WS60.C2\n",
      "Genome GenBank Accession Numbers:\n",
      "Toggle display:   \n",
      "Open fullscreen\n",
      "Genome accession\n",
      "Raw read accession\n",
      "BioProject\n",
      "BioSample.name\n",
      "CP162130-CP162132\n",
      "SAMN42533266\n",
      "PRJNA1136487\n",
      "Leptospira_gorisiae_WS92.C1\n",
      "CP162133-CP162136\n",
      "SAMN42533265\n",
      "PRJNA1136487\n",
      "Leptospira_milleri_WS60.C2\n",
      "CP162137-CP162138\n",
      "SAMN42533264\n",
      "PRJNA1136487\n",
      "Leptospira_cinconiae_WS58.C1\n",
      "CP162139-CP162141\n",
      "SAMN42533263\n",
      "PRJNA1136487\n",
      "Leptospira_mgodei_WS4.C2\n",
      "CP162142-CP162144\n",
      "SAMN42533262\n",
      "PRJNA1136487\n",
      "Leptospira_iowaensis_WS39.C2\n",
      " \n",
      "orgname ['Leptospira gorisiae ']\n",
      "accessions ['PQ066384', 'PQ066385', 'PQ066386', 'PQ066387', 'PQ066388', 'CP162137', 'CP162138', 'CP162130', 'CP162132', 'CP162142', 'CP162144', 'CP162139', 'CP162141', 'CP162133', 'CP162136', 'PQ066384', 'PQ066385', 'PQ066386', 'PQ066387', 'PQ066388', 'CP162130', 'CP162132', 'SAMN42533266', 'CP162133', 'CP162136', 'SAMN42533265', 'CP162137', 'CP162138', 'SAMN42533264', 'CP162139', 'CP162141', 'SAMN42533263', 'CP162142', 'CP162144', 'SAMN42533262']\n",
      "strain names [' WS92.C1T (=NVSL-WS92.C1T=KIT0303T']\n",
      "Description of Leptospira gorisiae sp. nov.\n",
      "orgname ['Leptospira gorisiae ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Four novel nontuberculous mycobacteria species: Mycobacterium wendilense sp. nov., Mycobacterium burgundiense sp. nov., Mycobacterium kokjensenii sp. nov. and Mycobacterium holstebronense sp. nov. revived from a historical Danish strain collection\n",
      "Tistrella arctica sp. nov., isolated from the surface seawater of the Bering Sea and Chukchi Sea and emended description of the genus Tistrella\n",
      "Luteimonas salinilitoris sp. nov., isolated from the shore soil of saline lake in Tibet of China\n",
      "Description of Luteimonas salinilitoris sp. nov\n",
      " Luteimonas salinilitoris (sa.li.ni.li'to.ris. L. masc. adj. salinus, salty; L. neut. n. litor, shore, coast; N.L. gen. n. salinilitoris, of or coming from a salty shore)\n",
      "Cells are Gram-stain-negative, aerobic, 0.500.53 m wide and 2.543.96 m long. Colonies are yellow, circular and convex after being cultivated on MA at 30  for 72 h. Growth is observed at temperatures ranging from 4 to 35 (optimum,30), in the presence of 08.0% (w/v) NaCl (optimum, 6.0%) and at pH levels of 6.010.0 (optimum, pH 7.0). The strain is positive for activities of oxidase but negative for catalase activity. Strain is able to hydrolyse Tween 20 but unable to hydrolyse Tween 80, starch, protease and cellulase. In the API ZYM system, it is positive for the activities of alkaline phosphatase, esterase (C4), lipid esterase (C8), leucine arylamidase, valine arylamidase, cystine arylamidase, trypsin, chymotrypsin, acid phosphatase, naphthol-AS-BI-phosphohydrolase, -galactosidase, -galactosidase, -glucosidase, -glucosidase, N-acetyl--glucosaminidase and -mannosidase, but negative for the activities of lipase (C14). In the API 20NE test, it is positive for the activity of aesculin, gelatin hydrolysis and -galactosidase, but negative for the activities of arginine dihydrolase, indole production, nitrate reduction and glucose fermentation. It utilizes dextrin, d-maltose, d-trehalose, d-cellobiose, gentiobiose, turanose, d-raffinose, -d-lactose, melibiose, d-salicylin, N-acetyl-d-glucosamine, N-acetyl-d-galactosamine, -d-glucose, d-mannose, d-galactose, l-fucose, inositol, gelatin, l-glutamic acid, -methyl-d-glucoside, d-fructose, d-fucose, Tween 40, -hydroxy-d, l-butyric acid, -butanoic acid, acetoacetate and propionic acid as sole carbon sources. The ubiquinone is Q-8. The major cellular fatty acids are iso-C11: 0, iso-C15: 0, iso-C16: 0 and iso-C17:1 9c. The major polar lipids are diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine and two unidentified phospholipids.\n",
      "The type strain C3-2-a3T (CGMCC 1.14507T = KCTC 8642T), isolated from a soil sample of LungmuCo Lake in Tibet, PR China. The genomic DNA G+C content of type strain is 68.39%. The 16S rRNA gene sequence and genome accession numbers for strains C3-2-a3T and B3-2-R+30 are PQ065621 and PQ151652, JBFWIB000000000 and JBFWIC000000000, respectively.\n",
      " \n",
      "orgname ['Luteimonas salinilitoris ']\n",
      "accessions ['PQ065621', 'PQ151652', 'JBFWIB000000000', 'JBFWIC000000000']\n",
      "strain names [' C3-2-a3T (CGMCC 1.14507T = KCTC 8642T']\n",
      "Description of Luteimonas salinilitoris sp. nov\n",
      "orgname ['Luteimonas salinilitoris ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Tenacibaculum platacis sp. nov., Tenacibaculum vairaonense sp. nov. and Tenacibaculum polynesiense sp. nov. isolated from batfish (Platax orbicularis) in Tahiti Island, French Polynesia\n",
      "Dyella aluminiiresistens sp. nov., a Al3+-tolerant bacterium with the ability to inhibit Fusarium oxysporum f. sp. melonis isolated from rhizosphere soil of muskmelon\n",
      "Description of Dyella aluminiiresistens sp. nov.\n",
      " Dyella aluminiiresistens (a.lu.mi.ni.i.re.sistens. N.L. neut. n. aluminium, aluminium; L. pres. part. resistens, resisting; N.L. part. adj. aluminiiresistens, resisting aluminium).\n",
      "Colonies are light yellow, circular and transparent on R2A plates after 3 days of incubation at 30. The cells are Gram-negative, non-spore-forming, aerobic, short rod (0.40.51.52.0 m) and motile with a single polar flagellum. Oxidase and catalase are negative. Growth is observed at 1537  (optimum, 30 ), pH 4.58.0 (optimum, pH 6.5) and NaCl concentrations of 03.0% (w/v, optimum 0.5%). In the API 20NE test, it is positive for nitrate reduction, aesculin hydrolysis, 4-nitrophenyl--d-galactopyranoside and assimilation of d-glucose but negative for indole production, d-glucose fermentation, arginine dihydrolase, hydrolysis of urea and gelatin, assimilation of l-arabinose, d-mannose, d-mannitol, N-acetyl-glucosamine, d-maltose, potassium gluconate, capric acid, adipic acid, malic acid, trisodium citrate and phenylacetic acid. In the API ZYM test, it is positive for activities of esterase (C4), esterase lipase (C8), leucine arylamidase, valine arylamidase, acid phosphatase, naphthol-AS-BI-phosphohydrolase, -galactosidase and -glucosidase but negative for alkaline phosphatase, lipase (C14), cystine arylamidase, trypsin, -chymotrypsin, -galactosidase, -glucuronidase, -glucosidase, N-acetyl--glucosaminidase, -mannosidase and -fucosidase. In the API 50CH test, acid production is observed with the following compounds: d-glucose (weakly positive), aesculin, d-maltose (weakly positive), d-sucrose and d-trehalose. The only respiratory quinone is ubiquinone-8 (Q-8), and the major fatty acids (5%) are iso-C17:0, C16:0, summed feature 9 (iso-C17:1 9c and/or C16:0 10-methyl), iso-C15:0, iso-C16:0 and anteiso-C17:0. The polar lipid profile consists of DPG, PG, PE, two unidentified aminophospholipid (APL1-2) and three unidentified phospholipids (PL1-3). The DNA G+C content is 65.1 %.\n",
      "The type strain, A6T (= GDMCC 1.4640T = KCTC 92542T), was isolated from the rhizosphere soil of a healthy muskmelon in Wenchang, Hainan Province, Peoples Republic of China (110.46 E, 19.32 N). The GenBank accession numbers for the 16S rRNA gene sequence and genome sequence accession of strain A6T are OR206507 and CP132911, respectively.\n",
      " \n",
      "orgname ['Dyella aluminiiresistens ']\n",
      "accessions ['OR206507', 'CP132911']\n",
      "strain names [', A6T (= GDMCC 1.4640T = KCTC 92542T']\n",
      "Description of Dyella aluminiiresistens sp. nov.\n",
      "orgname ['Dyella aluminiiresistens ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Emendation of Recommendation 6(7), Rule 64 and Appendix 9 Section D of the International Code of Nomenclature of Prokaryotes to regulate the formation of prokaryote names from personal names\n",
      "Chionosphaera pinicorticola sp. nov., a novel basidiomycetous yeast species isolated from pine tree bark in Gyeongju, South Korea\n",
      "International Committee on Systematics of Prokaryotes: minutes of the open plenary meeting, Tuesday, 22 October 2024, Sesto Fiorentino, Florence, Italy, and via Teams\n",
      "Pseudonocardia spirodelae sp. nov., isolated from duckweed and formal proposal to reclassify Pseudonocardia antarctica as a later heterotypic synonym of Pseudonocardia alni and reclassify Pseudonocardia carboxydivorans as Pseudonocardia alni subsp. carboxydivorans\n",
      "Description of Pseudonocardia spirodelae sp. nov.\n",
      " Pseudonocardia spirodelae (spi.ro.delae. N.L. gen. n. spirodelae, pertaining to Spirodela, the genus of the host plant).\n",
      "Aerobic, Gram-stain-positive actinobacterium that forms extensively branched substrate mycelium and aerial mycelium that fragmented into rod-shaped spores, with smooth surfaces. Yellowish-white aerial mycelium and dark orange-yellow substrate mycelium are formed on ISP 2 agar. No soluble pigments are produced on all tested agar media. They grow well on ISP media 1, 2, 3, 4, 6 and 7; GYE agar; MBA; CZA; NA; and YSA. There is moderate growth on ISP 5 medium. There is no melanin production. Growth occurs at a temperature range of 1739 C (optimum, 2530 C), pH 6.012.0 (optimum, pH 6.08.0) and NaCl concentration of 011% (w/v) (optimum, 07%). It is positive for catalase, but negative for oxidase. Nitrate reduction and H2S production are negative. No growth occurs in the presence of 0.005% (w/v) lysozyme. Aesculin, hypoxanthine, l-tyrosine, starch, Tween 20 and Tween 80 are degraded, but not arbutin, CMC, gelatin, guanine, skimmed milk, urea, xanthine and xylan. Adonitol, l-arabinose, d-dextrose, d-fructose, d-galactose, d-glucose, glycerol, d-mannitol, d-mannose, d-rhamnose, d-sorbitol, sucrose, sodium citrate, sodium acetate, sodium propionate, d-xylose and xylitol are utilized as sole carbon sources, but d-cellobiose, d-lactose, maltose, myo-inositol, d-raffinose, d-ribose and trehalose are not utilized. l-Asparagine, glycine and KNO3 are utilized as sole nitrogen sources, but histidine, methionine and hydroxy proline are not utilized, containing meso-diaminopimelic acid in whole-cell hydrolysates. The major polar lipids are phosphatidylethanolamine, phosphatidyl-choline and phosphatidylglycerol. Whole-cell sugars are arabinose, galactose, glucose and trace amounts of ribose. Mycolic acids are absent. The MK is MK-8(H4). The predominant cellular fatty acids (>5%) were iso-C16:0, iso-C16:1 H, summed feature 3: C16:1 7c/C16:1 6c; C16:1 6c/C16:1 7c, C17:1 8c and anteiso-C17:0.\n",
      "The type strain is DW16-2T (= TBRC 16418T = NBRC 115857T), which was isolated from duckweed (S. polyrhiza), collected from a garden pond in Chumphon Province, Thailand. The DNA G+C content of the type strain is 75.2%. The GenBank accession numbers of the genome and 16S rRNA gene of strain DW16-2T are JBBJUP000000000 and PP738009, respectively.\n",
      " \n",
      "orgname ['Pseudonocardia spirodelae ']\n",
      "accessions ['JBBJUP000000000', 'PP738009']\n",
      "strain names [' is DW16-2T (= TBRC 16418T = NBRC 115857T']\n",
      "Description of Pseudonocardia spirodelae sp. nov.\n",
      "orgname ['Pseudonocardia spirodelae ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Pseudonocardia alni subsp. alni subsp. nov.\n",
      " Pseudonocardia alni subsp. alni (alni. L. gen. fem. n. alni, of alder, referring to the isolation of the type strain and some other strains from alder associations).\n",
      "Yellowish-grey aerial mycelium and dark orange-yellow substrate mycelium are formed on ISP 2 agar, while a light reddish-brown soluble pigment is produced on ISP 7 agar. Growth occurs at a temperature range of 1337 C, pH 6.012.0 and NaCl concentration of 013% (w/v). Aesculin, arbutin, gelatin, l-tyrosine, starch, urea and xanthine are degraded, but not hypoxanthine. l-Arabinose, d-cellobiose, d-galactose, maltose, d-sorbitol, sodium propionate, trehalose, d-xylose and xylitol are utilized as sole carbon sources, but d-lactose, myo-inositol, d-rhamnose, d-ribose or sodium acetate are not utilized. Glycine and histidine are utilized as sole nitrogen sources.\n",
      "The G+C content of the type-strain genome is 74.0%, with an approximate size of 6.0 Mbp. The accession numbers of the genome and 16S rRNA gene of strain P. alni subsp. alni DSM 44104T are JBBJUP000000000 and Y08535, respectively.\n",
      "The type strain is DSM 44104T (= 3 LST = IFO 14991T = JCM 9103T=NBRC 14991T = VKM Ac-901T).\n",
      " \n",
      "orgname ['Pseudonocardia alni subsp. alni ']\n",
      "accessions ['JBBJUP000000000']\n",
      "strain names [' and some other strains from alder associations', ' is DSM 44104T (= 3 LST = IFO 14991T = JCM 9103T=NBRC 14991T = VKM Ac-901T']\n",
      "Description of Pseudonocardia alni subsp. alni subsp. nov.\n",
      "orgname ['Pseudonocardia alni subsp. alni ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n",
      "Description of Pseudonocardia alni subsp. carboxydivorans subsp. nov.\n",
      " Pseudonocardia alni subsp. carboxydivorans (car.bo.xydi.vorans. N.L. neut. adj. carboxydum, carbon monoxide; L. pres. part. vorans, devouring, digesting; N.L. fem. part. adj. carboxydivorans, digesting carbon monoxide).\n",
      "Yellowish-grey aerial mycelium and light yellowish-brown substrate mycelium are formed on ISP 2 agar. No soluble pigments are produced on all tested agar media. Growth occurs at a temperature range of 1537 C, pH 6.011.0 and NaCl concentration of 013% (w/v). Aesculin, arbutin, gelatin, hypoxanthine, l-tyrosine, starch and xanthine are degraded, but not urea. d-Galactose, maltose, d-ribose, d-sorbitol, sodium propionate, trehalose, d-xylose and xylitol are utilized as sole carbon sources, but l-arabinose, d-cellobiose, d-lactose, myo-inositol, d-rhamnose or sodium acetate are not utilized. Glycine is utilized as the sole nitrogen source, but histidine is not utilized.\n",
      "The G+C content of the type-strain genome is 74.5%, with an approximate size of 5.8 Mbp. The accession numbers of the genome of strain P. alni subsp. carboxydivorans JCM 14827T is JBBPIX000000000.\n",
      "The type strain is JCM 14827T (= Y8T = SWP-2006T = KCCM 42678T = DSM 45349T).\n",
      " \n",
      "orgname ['Pseudonocardia alni subsp. carboxydivorans ']\n",
      "accessions ['JBBPIX000000000']\n",
      "strain names [' is JCM 14827T (= Y8T = SWP-2006T = KCCM 42678T = DSM 45349T']\n",
      "Description of Pseudonocardia alni subsp. carboxydivorans subsp. nov.\n",
      "orgname ['Pseudonocardia alni subsp. carboxydivorans ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Genome-based reclassification of Suonthocola fibrivorans as a later heterotypic synonym of Diplocloster agilis\n",
      "Vishniacozyma siamensis sp. nov., a new anamorphic tremellomycetous yeast species isolated from a mangrove forest in Thailand\n",
      "Description of Vishniacozyma siamensis Gungprakhon, Khammeankae, Limtong and Khunnamwong sp. nov\n",
      " Vishniacozyma siamensis (si.am.ensis. N.L. fem. adj. siamensis, of or belonging to Siam, the old name of Thailand, where the type strain was isolated).\n",
      "After 7 days at 25 C on YM agar, the colony is cream-coloured, convex and has an entire margin (Fig. 2a, b). The yeast cells are subglobose to ovoid (1.21.8 m1.62.9 m). Multilateral budding is observed (Fig. 2c). Pseudohyphae are not formed in slide culture on PDA after 7 days at 25 C. Ballistospore formation is not observed on YM agar, 5% malt extract agar and corn meal agar after 14 days at 25 C. Sexual reproduction is not observed on PDA, corn meal agar, 5% malt extract agar, YPD agar, YM agar, Fowells acetate agar and Gorodkowa agar at 15 and 25 C after 4 weeks. Fermentation of glucose is negative. d-Glucose, d-galactose, l-arabinose, l-rhamnose, methyl -d-glycoside, salicin, succinate, erythritol, d-gluconate, xylitol, d-ribose (or delayed positive), glycerol, ethanol (variable), l-sorbose, d-arabinose, inulin, ribitol, dl-lactate (positive or weak), N-acetyl glucosamine, d-galacturonic acid (or weak), d-glucitol, d-glucono-1,5-lactone, myo-inositol (or slow positive), d-xylose, sucrose, maltose, trehalose, cellobiose, melibiose, melizitose, lactose, raffinose, d-mannitol, galactitol are assimilated, but d-glucuronate, citrate, are not. Nitrogen assimilation is positive for ammonium sulphate, ethylamine, cadaverine and l-lysine, but slowly positive for potassium nitrate and slowly or delayed positive for sodium nitrite. Growth in a vitamin-free medium is positive. Growth occurs on media containing 50 and 60% glucose, and 10% sodium chloride/5% glucose is weakly positive. There is no growth in the presence of 0.01 and 0.1% cycloheximide, as well as 15% sodium chloride/5% glucose. Growth is observed at 1530 C, but not at 35 C. Soluble starch-like extracellular carbohydrates are not produced. Acid formation is negative. Urea hydrolysis and the DBB reaction are positive.\n",
      "Fig. 2.\n",
      "Vishniacozyma siamensis sp. nov. TBRC 18499T (a, b) Colonies on YM agar after 7 days at 25 C (bar, 2 mm) and (c) Yeast cells on YM agar after 3 days at 25 C (bar, 10 m)\n",
      "\n",
      "\n",
      "Click to view\n",
      " TBRC 18499 is designated as a holotype of V. siamensis sp. nov. It was isolated from the green leaves of Rhizophora apiculata collected from the International Mangrove Botanical Garden Rama IX, Chanthaburi province, Thailand. It is permanently preserved in a metabolically inactive state in the Thailand Bioresource Research Center (TBRC), National Center for Genetic Engineering and Biotechnology, Thailand. An isotype culture has been preserved in a metabolically inactive state in the Portuguese Yeast Culture Collection, as PYCC 10042. The MycoBank number for V. siamensis is MB 855838. The GenBank/EMBL/DDBJ/PIR accession numbers for the ITS regions of strains DMKU-SG26, DMKU-RGM25, DMKU-SG42, DMKU-SYM22, DMKU-RG41 and DMKU-RX317 are LC786857, LC832838, LC832837, LC832840, LC832839 and LC818861, respectively, and those for D1/D2 domains of the LSU rRNA genes are LC797015, LC832841, LC797016, LC832842, LC832843 and LC832855, respectively.\n",
      " \n",
      "orgname ['and Khunnamwong ', 'Vishniacozyma siamensis ', 'V. siamensis ']\n",
      "accessions ['LC786857', 'LC832838', 'LC832837', 'LC832840', 'LC832839', 'LC818861', 'LC797015', 'LC832841', 'LC797016', 'LC832842', 'LC832843', 'LC832855']\n",
      "strain names [' was isolated']\n",
      "Description of Vishniacozyma siamensis Gungprakhon, Khammeankae, Limtong and Khunnamwong sp. nov\n",
      "orgname ['and Khunnamwong ']\n",
      "accessions []\n",
      "strain names []\n",
      "BREAK\n"
     ]
    }
   ],
   "source": [
    "# Remove non-ASCII characters\n",
    "def remove_non_ascii(text):\n",
    "    return ''.join(char for char in text if ord(char) < 128)\n",
    "\n",
    "\n",
    "pub_df = pd.DataFrame(columns=['PublishedName', 'Accessions', 'Strains', 'Authority', 'DOI', 'URL'])\n",
    "pd.set_option('display.max_columns', None)\n",
    "combined_description = []\n",
    "for url in urls:\n",
    "    counter = 1\n",
    "    strains = []\n",
    "    accessions = []\n",
    "    doi = []\n",
    "    author = []\n",
    "    date = []\n",
    "    year = []\n",
    "    author_raw = []\n",
    "    author = []\n",
    "    name1 = []\n",
    "    name2 = []\n",
    "    author_count = []\n",
    "    authority = []\n",
    "    description = None\n",
    "    snumber = []    \n",
    "    #scrape_with_selenium(url, counter)\n",
    "    #scrape_with_beautifulsoup(url)\n",
    "    options = Options()\n",
    "    options.headless = False  # Set to True for headless mode\n",
    "    driver = webdriver.Chrome(options=options)\n",
    "    (driver.page_source).encode('utf-8')\n",
    "    #(driver.page_source).encode('ascii', 'ignore')\n",
    "\n",
    "    #Navigate to the webpage\n",
    "    driver.get(url)\n",
    "\n",
    "    #Allow time for dynamic content to load (you may need to use WebDriverWait for more robust waiting)\n",
    "    time.sleep(3)\n",
    "    \n",
    "    for element in driver.find_elements(By.CLASS_NAME, \"item-meta-data__item-title\"):\n",
    "        #print(element.text)\n",
    "        title = element.text\n",
    "        print(title)\n",
    "        \n",
    "    for element in driver.find_elements(By.PARTIAL_LINK_TEXT, \"doi.org\"):\n",
    "        doi = element.text\n",
    "        #print(doi)\n",
    "    \n",
    "    #find publication year\n",
    "    for element in driver.find_elements(By.XPATH, \"//*[@id='bellowheadercontainer']/main/div[2]/div/ul/li[3]/span/span[2]\"):\n",
    "        date = element.text\n",
    "        year = date[-4:]\n",
    "        #print(year)\n",
    "    \n",
    "    #find authors\n",
    "    for element in driver.find_elements(By.XPATH, \"//*[@id='bellowheadercontainer']/main/div[2]/div/ul/li[1]/span\"):\n",
    "        author_raw = element.text\n",
    "        #print(author_raw)\n",
    "        author = re.sub(r\"[\\d+]+\",'',author_raw)\n",
    "        author = re.sub(r\"†\",'',author)\n",
    "        author = re.sub(r\" and \",',',author)\n",
    "        author = re.sub(r\",,\",',',author)  \n",
    "        author = author.split(',')\n",
    "        author[:] = [item for item in author if item != '']\n",
    "        #print(author)\n",
    "        author_count = len(author)\n",
    "        #print(author_count)\n",
    "        if author_count == 1:\n",
    "            name1 = (str(author[0])) \n",
    "            name1 = HumanName(name1)\n",
    "            authority = name1.last + ' ' +str(year)\n",
    "            #print(authority)\n",
    "        elif author_count ==2:\n",
    "            name1 = (str(author[0]))\n",
    "            name1 = HumanName(name1)\n",
    "            name2 = (str(author[1]))\n",
    "            name2 = HumanName(name2)\n",
    "            authority = name1.last + ' and ' + name2.last + ' ' +str(year)\n",
    "            #print(authority)\n",
    "        else:\n",
    "            name1 = (str(author[0]))\n",
    "            name1 = HumanName(name1)\n",
    "            authority = name1.last + ' et al. ' +str(year)\n",
    "            #print(authority)\n",
    "    \n",
    "    #extract data from the species description\n",
    "    for element in driver.find_elements(By.CSS_SELECTOR, \"div.tl-main-part.title\"): #finds section headers\n",
    "        #print(element.text)\n",
    "        counter += 1\n",
    "        description = element.text\n",
    "        #print(description)\n",
    "        if \"Description of\" in description: \n",
    "            #print('found', description)  \n",
    "            #snumber = 's' + str(counter - 4) + '/p[3]'\n",
    "            snumber = 's' + str(counter - 4)\n",
    "            #print('snumber is', snumber)\n",
    "            for element in driver.find_elements(By.ID, snumber):\n",
    "                description = element.text\n",
    "                description = description.replace('\\u2009', ' ')\n",
    "                cleaned_text = remove_non_ascii(description)\n",
    "                print(cleaned_text)\n",
    "                combined_description.append(cleaned_text)\n",
    "        \n",
    "                #find the organism names    \n",
    "                match = [r'(\\S+\\s+){2}(?=sp. nov)', r'(\\S+\\s+){2}(?=nom. nov.)', r'(\\S+\\s+){2}(?=SP. NOV.)', r'(\\S+\\s+){4}(?=subsp. nov.)']\n",
    "                regex = re.compile(r'\\b(' + '|'.join(match) + r')\\b')\n",
    "                orgname = [m.group() for m in regex.finditer(cleaned_text)]\n",
    "                print('orgname', orgname)\n",
    "\n",
    "                #find the accessions\n",
    "                pattern = [r'[A-Z]{2}\\d{6}', r'[A-Z]{4}\\d{8}', r'([A-Z]+)(_[A-Z]+)\\d{6}', r'[A-Z]{6}\\d{9}']\n",
    "                regex = re.compile(r'\\b(' + '|'.join(pattern) + r')\\b')\n",
    "                if description is not None:\n",
    "                    accessions = [m.group() for m in regex.finditer(cleaned_text)]\n",
    "                    print('accessions', accessions)\n",
    "    \n",
    "                #find the strains\n",
    "                #strainpattern = [r'(?<=type strain).*'] \n",
    "                strainpattern = [r'(?<=type strain).*?(?=\\))']\n",
    "                regex = re.compile(r'\\b(' + '|'.join(strainpattern) + r')\\b')\n",
    "                if description is not None:\n",
    "                    strains = [m.group() for m in regex.finditer(cleaned_text)]\n",
    "                    print('strain names', strains)\n",
    "    \n",
    "                #load data into pandas dataframe\n",
    "                row_data = [orgname, accessions, strains, authority, doi, url]\n",
    "                length = len(pub_df)\n",
    "                pub_df.loc[length] = row_data\n",
    "            print('BREAK')\n",
    "    \n",
    "#Close the browser window\n",
    "    driver.quit()    \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8a2b16f8",
   "metadata": {},
   "outputs": [],
   "source": [
    "#write description to a file\n",
    "#print(combined_description)\n",
    "file = open('all_descriptions29', \"w\")\n",
    "file.writelines(combined_description)\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "42783410",
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('max_colwidth', None)\n",
    "pub_df['Strains'] = [','.join(map(str, l)) for l in pub_df['Strains']]\n",
    "#pub_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "2d240cfe",
   "metadata": {},
   "outputs": [],
   "source": [
    "pub_df = pub_df.drop_duplicates(subset='PublishedName', keep=\"first\")\n",
    "#pub_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "d6a00bb9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PublishedName</th>\n",
       "      <th>Accessions</th>\n",
       "      <th>Strains</th>\n",
       "      <th>Authority</th>\n",
       "      <th>DOI</th>\n",
       "      <th>URL</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Roseomonas cutis</td>\n",
       "      <td>[OP788068]</td>\n",
       "      <td>is OT10T (=KCTC 92087T =JCM 34968T, is 72.6 mol% (genome</td>\n",
       "      <td>Son et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006617</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006617%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018546452%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=qPgbRuHApRO85zY1%2FXyHK7fwpRhZIQDtoO%2BMyTwMQ8I%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>and Baghela</td>\n",
       "      <td>[MB855459]</td>\n",
       "      <td></td>\n",
       "      <td>Sipiczki and Baghela 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006629</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006629%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018597448%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=gTvoT9taoY5EywTBMznivistMD3RQuxyiE0TB92Vbu8%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Methylobacterium litchii</td>\n",
       "      <td>[PP600109, JBBXMY000000000]</td>\n",
       "      <td>is SD21T (=GDMCC 1.4327T=KCTC 8300T</td>\n",
       "      <td>Gao et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Methylobacterium guangdongense</td>\n",
       "      <td>[PP600110, JBBXMX000000000]</td>\n",
       "      <td>is SI9T (=GDMCC 1.14329T= KCTC 8298T</td>\n",
       "      <td>Gao et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Methylorubrum subtropicum</td>\n",
       "      <td>[PP600111, JBBXMW000000000]</td>\n",
       "      <td>is SB2T (=GDMCC 1.14328T= KCTC 8299T</td>\n",
       "      <td>Gao et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Pseudomonas fungipugnans</td>\n",
       "      <td>[OQ547267, JARBWL000000000]</td>\n",
       "      <td></td>\n",
       "      <td>Schnyder et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>P. fungipugnans</td>\n",
       "      <td>[OQ547267, JARBWL000000000]</td>\n",
       "      <td></td>\n",
       "      <td>Schnyder et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Pseudomonas fungipugnans</td>\n",
       "      <td>[]</td>\n",
       "      <td></td>\n",
       "      <td>Schnyder et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Leptospira gorisiae</td>\n",
       "      <td>[PQ066384, PQ066385, PQ066386, PQ066387, PQ066388, CP162137, CP162138, CP162130, CP162132, CP162142, CP162144, CP162139, CP162141, CP162133, CP162136, PQ066384, PQ066385, PQ066386, PQ066387, PQ066388, CP162130, CP162132, SAMN42533266, CP162133, CP162136, SAMN42533265, CP162137, CP162138, SAMN42533264, CP162139, CP162141, SAMN42533263, CP162142, CP162144, SAMN42533262]</td>\n",
       "      <td>WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td>Hamond et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Luteimonas salinilitoris</td>\n",
       "      <td>[PQ065621, PQ151652, JBFWIB000000000, JBFWIC000000000]</td>\n",
       "      <td>C3-2-a3T (CGMCC 1.14507T = KCTC 8642T</td>\n",
       "      <td>Zhang et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006630</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Dyella aluminiiresistens</td>\n",
       "      <td>[OR206507, CP132911]</td>\n",
       "      <td>, A6T (= GDMCC 1.4640T = KCTC 92542T</td>\n",
       "      <td>Li et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006611</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006611%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018853339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=S4R1T9Bs6FJngYjedz8T7GqhIOrzYpyrq3%2BkVjxy5%2Bw%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Pseudonocardia spirodelae</td>\n",
       "      <td>[JBBJUP000000000, PP738009]</td>\n",
       "      <td>is DW16-2T (= TBRC 16418T = NBRC 115857T</td>\n",
       "      <td>Butdee et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Pseudonocardia alni subsp. alni</td>\n",
       "      <td>[JBBJUP000000000]</td>\n",
       "      <td>and some other strains from alder associations, is DSM 44104T (= 3 LST = IFO 14991T = JCM 9103T=NBRC 14991T = VKM Ac-901T</td>\n",
       "      <td>Butdee et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Pseudonocardia alni subsp. carboxydivorans</td>\n",
       "      <td>[JBBPIX000000000]</td>\n",
       "      <td>is JCM 14827T (= Y8T = SWP-2006T = KCCM 42678T = DSM 45349T</td>\n",
       "      <td>Butdee et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>and Khunnamwong</td>\n",
       "      <td>[LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]</td>\n",
       "      <td>was isolated</td>\n",
       "      <td>Gungprakhon et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Vishniacozyma siamensis</td>\n",
       "      <td>[LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]</td>\n",
       "      <td>was isolated</td>\n",
       "      <td>Gungprakhon et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>V. siamensis</td>\n",
       "      <td>[LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]</td>\n",
       "      <td>was isolated</td>\n",
       "      <td>Gungprakhon et al. 2025</td>\n",
       "      <td>https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td>https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&amp;data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&amp;sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&amp;reserved=0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  PublishedName  \\\n",
       "0                             Roseomonas cutis    \n",
       "1                                  and Baghela    \n",
       "2                     Methylobacterium litchii    \n",
       "3               Methylobacterium guangdongense    \n",
       "4                    Methylorubrum subtropicum    \n",
       "5                     Pseudomonas fungipugnans    \n",
       "6                              P. fungipugnans    \n",
       "7                     Pseudomonas fungipugnans    \n",
       "8                          Leptospira gorisiae    \n",
       "9                     Luteimonas salinilitoris    \n",
       "10                    Dyella aluminiiresistens    \n",
       "11                   Pseudonocardia spirodelae    \n",
       "12             Pseudonocardia alni subsp. alni    \n",
       "13  Pseudonocardia alni subsp. carboxydivorans    \n",
       "14                             and Khunnamwong    \n",
       "15                     Vishniacozyma siamensis    \n",
       "16                                V. siamensis    \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                            Accessions  \\\n",
       "0                                                                                                                                                                                                                                                                                                                                                                           [OP788068]   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                           [MB855459]   \n",
       "2                                                                                                                                                                                                                                                                                                                                                          [PP600109, JBBXMY000000000]   \n",
       "3                                                                                                                                                                                                                                                                                                                                                          [PP600110, JBBXMX000000000]   \n",
       "4                                                                                                                                                                                                                                                                                                                                                          [PP600111, JBBXMW000000000]   \n",
       "5                                                                                                                                                                                                                                                                                                                                                          [OQ547267, JARBWL000000000]   \n",
       "6                                                                                                                                                                                                                                                                                                                                                          [OQ547267, JARBWL000000000]   \n",
       "7                                                                                                                                                                                                                                                                                                                                                                                   []   \n",
       "8   [PQ066384, PQ066385, PQ066386, PQ066387, PQ066388, CP162137, CP162138, CP162130, CP162132, CP162142, CP162144, CP162139, CP162141, CP162133, CP162136, PQ066384, PQ066385, PQ066386, PQ066387, PQ066388, CP162130, CP162132, SAMN42533266, CP162133, CP162136, SAMN42533265, CP162137, CP162138, SAMN42533264, CP162139, CP162141, SAMN42533263, CP162142, CP162144, SAMN42533262]   \n",
       "9                                                                                                                                                                                                                                                                                                                               [PQ065621, PQ151652, JBFWIB000000000, JBFWIC000000000]   \n",
       "10                                                                                                                                                                                                                                                                                                                                                                [OR206507, CP132911]   \n",
       "11                                                                                                                                                                                                                                                                                                                                                         [JBBJUP000000000, PP738009]   \n",
       "12                                                                                                                                                                                                                                                                                                                                                                   [JBBJUP000000000]   \n",
       "13                                                                                                                                                                                                                                                                                                                                                                   [JBBPIX000000000]   \n",
       "14                                                                                                                                                                                                                                                            [LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]   \n",
       "15                                                                                                                                                                                                                                                            [LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]   \n",
       "16                                                                                                                                                                                                                                                            [LC786857, LC832838, LC832837, LC832840, LC832839, LC818861, LC797015, LC832841, LC797016, LC832842, LC832843, LC832855]   \n",
       "\n",
       "                                                                                                                       Strains  \\\n",
       "0                                                                     is OT10T (=KCTC 92087T =JCM 34968T, is 72.6 mol% (genome   \n",
       "1                                                                                                                                \n",
       "2                                                                                          is SD21T (=GDMCC 1.4327T=KCTC 8300T   \n",
       "3                                                                                         is SI9T (=GDMCC 1.14329T= KCTC 8298T   \n",
       "4                                                                                         is SB2T (=GDMCC 1.14328T= KCTC 8299T   \n",
       "5                                                                                                                                \n",
       "6                                                                                                                                \n",
       "7                                                                                                                                \n",
       "8                                                                                            WS92.C1T (=NVSL-WS92.C1T=KIT0303T   \n",
       "9                                                                                        C3-2-a3T (CGMCC 1.14507T = KCTC 8642T   \n",
       "10                                                                                        , A6T (= GDMCC 1.4640T = KCTC 92542T   \n",
       "11                                                                                    is DW16-2T (= TBRC 16418T = NBRC 115857T   \n",
       "12   and some other strains from alder associations, is DSM 44104T (= 3 LST = IFO 14991T = JCM 9103T=NBRC 14991T = VKM Ac-901T   \n",
       "13                                                                 is JCM 14827T (= Y8T = SWP-2006T = KCCM 42678T = DSM 45349T   \n",
       "14                                                                                                                was isolated   \n",
       "15                                                                                                                was isolated   \n",
       "16                                                                                                                was isolated   \n",
       "\n",
       "                    Authority                                     DOI  \\\n",
       "0             Son et al. 2025  https://doi.org/10.1099/ijsem.0.006617   \n",
       "1   Sipiczki and Baghela 2025  https://doi.org/10.1099/ijsem.0.006629   \n",
       "2             Gao et al. 2025  https://doi.org/10.1099/ijsem.0.006639   \n",
       "3             Gao et al. 2025  https://doi.org/10.1099/ijsem.0.006639   \n",
       "4             Gao et al. 2025  https://doi.org/10.1099/ijsem.0.006639   \n",
       "5        Schnyder et al. 2025  https://doi.org/10.1099/ijsem.0.006624   \n",
       "6        Schnyder et al. 2025  https://doi.org/10.1099/ijsem.0.006624   \n",
       "7        Schnyder et al. 2025  https://doi.org/10.1099/ijsem.0.006624   \n",
       "8          Hamond et al. 2025  https://doi.org/10.1099/ijsem.0.006595   \n",
       "9           Zhang et al. 2025  https://doi.org/10.1099/ijsem.0.006630   \n",
       "10             Li et al. 2025  https://doi.org/10.1099/ijsem.0.006611   \n",
       "11         Butdee et al. 2025  https://doi.org/10.1099/ijsem.0.006608   \n",
       "12         Butdee et al. 2025  https://doi.org/10.1099/ijsem.0.006608   \n",
       "13         Butdee et al. 2025  https://doi.org/10.1099/ijsem.0.006608   \n",
       "14    Gungprakhon et al. 2025  https://doi.org/10.1099/ijsem.0.006623   \n",
       "15    Gungprakhon et al. 2025  https://doi.org/10.1099/ijsem.0.006623   \n",
       "16    Gungprakhon et al. 2025  https://doi.org/10.1099/ijsem.0.006623   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     URL  \n",
       "0       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006617%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018546452%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=qPgbRuHApRO85zY1%2FXyHK7fwpRhZIQDtoO%2BMyTwMQ8I%3D&reserved=0  \n",
       "1           https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006629%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018597448%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=gTvoT9taoY5EywTBMznivistMD3RQuxyiE0TB92Vbu8%3D&reserved=0  \n",
       "2       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0  \n",
       "3       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0  \n",
       "4       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0  \n",
       "5       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0  \n",
       "6       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0  \n",
       "7       https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0  \n",
       "8     https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0  \n",
       "9         https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&reserved=0  \n",
       "10      https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006611%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018853339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=S4R1T9Bs6FJngYjedz8T7GqhIOrzYpyrq3%2BkVjxy5%2Bw%3D&reserved=0  \n",
       "11  https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0  \n",
       "12  https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0  \n",
       "13  https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0  \n",
       "14        https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0  \n",
       "15        https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0  \n",
       "16        https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#try drop duplicate accessions here\n",
    "pub_df = pub_df.drop_duplicates(subset='Accessions', keep=\"first\")\n",
    "pub_df.explode(['PublishedName']).reset_index(drop=True)\n",
    "#pub_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "e74e265b",
   "metadata": {},
   "outputs": [],
   "source": [
    "pub2_df = pub_df.explode(['Accessions']).reset_index(drop=True)\n",
    "#pub2_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "66791f58",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "58034d28",
   "metadata": {},
   "outputs": [],
   "source": [
    "pub4_df = pub2_df.explode(['PublishedName']).reset_index(drop=True)\n",
    "pub4_df.rename(columns={'Accessions' : 'accession'}, inplace=True)\n",
    "#pub4_df = pub4_df.dropna()\n",
    "#pub4_df = pub4_df.drop_duplicates(subset='accession', keep=\"first\")\n",
    "pub4_df=pub4_df[pub4_df['accession'].isnull() | ~pub4_df[pub4_df['accession'].notnull()].duplicated(subset='accession',keep='first')]\n",
    "#pub4_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "4a103350",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_unique= pub4_df.drop_duplicates([\"accession\"], keep=\"first\")\n",
    "#df_unique = df_unique.dropna()\n",
    "#df_unique"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d7f80dd1",
   "metadata": {},
   "source": [
    "### Create a dataframe of unique accessions and look up NCBI taxonomy information of each accession with srcchk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "84a9da33",
   "metadata": {},
   "outputs": [],
   "source": [
    "#df_unique.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "8208be48",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_unique['accession'] = df_unique['accession'].astype('str') "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "1be92fbc",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('acclist', 'w') as f:\n",
    "    for text in df_unique['accession'].tolist():\n",
    "        f.write(text + '\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "a97a9150",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error: Malformatted ID \"nan\"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.system(\"/netopt/ncbi_tools64/bin/srcchk -i acclist -f taxname,taxid,strain -o acclist.taxdata\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "1f0aaf52",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>accession</th>\n",
       "      <th>NCBIname</th>\n",
       "      <th>taxid</th>\n",
       "      <th>strain</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>OP788068</td>\n",
       "      <td>Roseomonas sp. OT10</td>\n",
       "      <td>2897332.0</td>\n",
       "      <td>OT10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PP600109</td>\n",
       "      <td>Methylobacterium sp.</td>\n",
       "      <td>409.0</td>\n",
       "      <td>SD21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>JBBXMY000000000</td>\n",
       "      <td>Methylobacterium sp. SD21</td>\n",
       "      <td>3138810.0</td>\n",
       "      <td>SD21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PP600110</td>\n",
       "      <td>Methylobacterium sp.</td>\n",
       "      <td>409.0</td>\n",
       "      <td>SI9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>JBBXMX000000000</td>\n",
       "      <td>Methylobacterium sp. SI9</td>\n",
       "      <td>3138811.0</td>\n",
       "      <td>SI9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>PP600111</td>\n",
       "      <td>Methylorubrum sp.</td>\n",
       "      <td>2282524.0</td>\n",
       "      <td>SB2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>JBBXMW000000000</td>\n",
       "      <td>Methylorubrum sp. SB2</td>\n",
       "      <td>3138812.0</td>\n",
       "      <td>SB2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>OQ547267</td>\n",
       "      <td>Pseudomonas sp.</td>\n",
       "      <td>306.0</td>\n",
       "      <td>#681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>JARBWL000000000</td>\n",
       "      <td>Pseudomonas sp. 681</td>\n",
       "      <td>3024217.0</td>\n",
       "      <td>681</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>PQ066384</td>\n",
       "      <td>Leptospira sp. WS58.C1</td>\n",
       "      <td>3235173.0</td>\n",
       "      <td>WS58.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>PQ066385</td>\n",
       "      <td>Leptospira sp. WS92.C1</td>\n",
       "      <td>3235175.0</td>\n",
       "      <td>WS92.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>PQ066386</td>\n",
       "      <td>Leptospira sp. WS39.C2</td>\n",
       "      <td>3235171.0</td>\n",
       "      <td>WS39.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>PQ066387</td>\n",
       "      <td>Leptospira sp. WS4.C2</td>\n",
       "      <td>3235172.0</td>\n",
       "      <td>WS4.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>PQ066388</td>\n",
       "      <td>Leptospira sp. WS60.C2</td>\n",
       "      <td>3235174.0</td>\n",
       "      <td>WS60.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>CP162137</td>\n",
       "      <td>Leptospira sp. WS58.C1</td>\n",
       "      <td>3235173.0</td>\n",
       "      <td>WS58.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>CP162138</td>\n",
       "      <td>Leptospira sp. WS58.C1</td>\n",
       "      <td>3235173.0</td>\n",
       "      <td>WS58.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>CP162130</td>\n",
       "      <td>Leptospira sp. WS92.C1</td>\n",
       "      <td>3235175.0</td>\n",
       "      <td>WS92.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>CP162132</td>\n",
       "      <td>Leptospira sp. WS92.C1</td>\n",
       "      <td>3235175.0</td>\n",
       "      <td>WS92.C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>CP162142</td>\n",
       "      <td>Leptospira sp. WS39.C2</td>\n",
       "      <td>3235171.0</td>\n",
       "      <td>WS39.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>CP162144</td>\n",
       "      <td>Leptospira sp. WS39.C2</td>\n",
       "      <td>3235171.0</td>\n",
       "      <td>WS39.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>CP162139</td>\n",
       "      <td>Leptospira sp. WS4.C2</td>\n",
       "      <td>3235172.0</td>\n",
       "      <td>WS4.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>CP162141</td>\n",
       "      <td>Leptospira sp. WS4.C2</td>\n",
       "      <td>3235172.0</td>\n",
       "      <td>WS4.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>CP162133</td>\n",
       "      <td>Leptospira sp. WS60.C2</td>\n",
       "      <td>3235174.0</td>\n",
       "      <td>WS60.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>CP162136</td>\n",
       "      <td>Leptospira sp. WS60.C2</td>\n",
       "      <td>3235174.0</td>\n",
       "      <td>WS60.C2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>PQ065621</td>\n",
       "      <td>Luteimonas sp. C3_2_a3</td>\n",
       "      <td>3237697.0</td>\n",
       "      <td>C3_2_a3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>PQ151652</td>\n",
       "      <td>Luteimonas sp.</td>\n",
       "      <td>1873125.0</td>\n",
       "      <td>B3_2_R+30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>JBFWIB000000000</td>\n",
       "      <td>Luteimonas sp. C3_2_a3</td>\n",
       "      <td>3237697.0</td>\n",
       "      <td>C3_2_a3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>JBFWIC000000000</td>\n",
       "      <td>Luteimonas sp. B3_2_R+30</td>\n",
       "      <td>3239301.0</td>\n",
       "      <td>B3_2_R+30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>OR206507</td>\n",
       "      <td>Dyella sp.</td>\n",
       "      <td>1869338.0</td>\n",
       "      <td>A6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>CP132911</td>\n",
       "      <td>Dyella sp. A6</td>\n",
       "      <td>3069105.0</td>\n",
       "      <td>A6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>JBBJUP000000000</td>\n",
       "      <td>Pseudonocardia sp. DW16-2</td>\n",
       "      <td>3133431.0</td>\n",
       "      <td>DW16-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>PP738009</td>\n",
       "      <td>Pseudonocardia sp.</td>\n",
       "      <td>60912.0</td>\n",
       "      <td>DW16-2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>JBBPIX000000000</td>\n",
       "      <td>Pseudonocardia carboxydivorans</td>\n",
       "      <td>415010.0</td>\n",
       "      <td>JCM 14827</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>LC786857</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SG26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>LC832838</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RGM25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>LC832837</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SG42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>LC832840</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SYM22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>LC832839</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RG41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>LC818861</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RX317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>LC797015</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SG26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>LC832841</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RGM25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>LC797016</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SG42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>LC832842</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-SYM22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>LC832843</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RG41</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>LC832855</td>\n",
       "      <td>Vishniacozyma sp.</td>\n",
       "      <td>1902918.0</td>\n",
       "      <td>DMKU-RX317</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          accession                        NCBIname      taxid      strain\n",
       "0          OP788068             Roseomonas sp. OT10  2897332.0        OT10\n",
       "2          PP600109            Methylobacterium sp.      409.0        SD21\n",
       "3   JBBXMY000000000       Methylobacterium sp. SD21  3138810.0        SD21\n",
       "4          PP600110            Methylobacterium sp.      409.0         SI9\n",
       "5   JBBXMX000000000        Methylobacterium sp. SI9  3138811.0         SI9\n",
       "6          PP600111               Methylorubrum sp.  2282524.0         SB2\n",
       "7   JBBXMW000000000           Methylorubrum sp. SB2  3138812.0         SB2\n",
       "8          OQ547267                 Pseudomonas sp.      306.0        #681\n",
       "9   JARBWL000000000             Pseudomonas sp. 681  3024217.0         681\n",
       "11         PQ066384          Leptospira sp. WS58.C1  3235173.0     WS58.C1\n",
       "12         PQ066385          Leptospira sp. WS92.C1  3235175.0     WS92.C1\n",
       "13         PQ066386          Leptospira sp. WS39.C2  3235171.0     WS39.C2\n",
       "14         PQ066387           Leptospira sp. WS4.C2  3235172.0      WS4.C2\n",
       "15         PQ066388          Leptospira sp. WS60.C2  3235174.0     WS60.C2\n",
       "16         CP162137          Leptospira sp. WS58.C1  3235173.0     WS58.C1\n",
       "17         CP162138          Leptospira sp. WS58.C1  3235173.0     WS58.C1\n",
       "18         CP162130          Leptospira sp. WS92.C1  3235175.0     WS92.C1\n",
       "19         CP162132          Leptospira sp. WS92.C1  3235175.0     WS92.C1\n",
       "20         CP162142          Leptospira sp. WS39.C2  3235171.0     WS39.C2\n",
       "21         CP162144          Leptospira sp. WS39.C2  3235171.0     WS39.C2\n",
       "22         CP162139           Leptospira sp. WS4.C2  3235172.0      WS4.C2\n",
       "23         CP162141           Leptospira sp. WS4.C2  3235172.0      WS4.C2\n",
       "24         CP162133          Leptospira sp. WS60.C2  3235174.0     WS60.C2\n",
       "25         CP162136          Leptospira sp. WS60.C2  3235174.0     WS60.C2\n",
       "31         PQ065621          Luteimonas sp. C3_2_a3  3237697.0     C3_2_a3\n",
       "32         PQ151652                  Luteimonas sp.  1873125.0   B3_2_R+30\n",
       "33  JBFWIB000000000          Luteimonas sp. C3_2_a3  3237697.0     C3_2_a3\n",
       "34  JBFWIC000000000        Luteimonas sp. B3_2_R+30  3239301.0   B3_2_R+30\n",
       "35         OR206507                      Dyella sp.  1869338.0          A6\n",
       "36         CP132911                   Dyella sp. A6  3069105.0          A6\n",
       "37  JBBJUP000000000       Pseudonocardia sp. DW16-2  3133431.0      DW16-2\n",
       "38         PP738009              Pseudonocardia sp.    60912.0      DW16-2\n",
       "39  JBBPIX000000000  Pseudonocardia carboxydivorans   415010.0   JCM 14827\n",
       "40         LC786857               Vishniacozyma sp.  1902918.0   DMKU-SG26\n",
       "41         LC832838               Vishniacozyma sp.  1902918.0  DMKU-RGM25\n",
       "42         LC832837               Vishniacozyma sp.  1902918.0   DMKU-SG42\n",
       "43         LC832840               Vishniacozyma sp.  1902918.0  DMKU-SYM22\n",
       "44         LC832839               Vishniacozyma sp.  1902918.0   DMKU-RG41\n",
       "45         LC818861               Vishniacozyma sp.  1902918.0  DMKU-RX317\n",
       "46         LC797015               Vishniacozyma sp.  1902918.0   DMKU-SG26\n",
       "47         LC832841               Vishniacozyma sp.  1902918.0  DMKU-RGM25\n",
       "48         LC797016               Vishniacozyma sp.  1902918.0   DMKU-SG42\n",
       "49         LC832842               Vishniacozyma sp.  1902918.0  DMKU-SYM22\n",
       "50         LC832843               Vishniacozyma sp.  1902918.0   DMKU-RG41\n",
       "51         LC832855               Vishniacozyma sp.  1902918.0  DMKU-RX317"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "taxdata_file_name = (r'acclist.taxdata')    \n",
    "srcchk_df = pd.read_csv(taxdata_file_name, sep='\\t', index_col=None, low_memory=False)\n",
    "srcchk_df.drop(columns=['Unnamed: 4'], inplace=True)\n",
    "srcchk_df.rename(columns={'organism' : 'NCBIname'}, inplace=True)\n",
    "srcchk_df['accession'] = srcchk_df['accession'].astype(str).replace('\\.\\d+', '', regex=True).astype(str)\n",
    "srcchk_df = srcchk_df.dropna()\n",
    "srcchk_df \n",
    "#many need to add dropna here\n",
    "#If /strain is not populated on the NCBI accession, we lose the NCBI srcchk info here"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "81b7fe78",
   "metadata": {},
   "source": [
    "### Combine dataframes into one"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "7f05118d",
   "metadata": {},
   "outputs": [],
   "source": [
    "combine_df=pd.merge(left=pub4_df, right=srcchk_df, left_on='accession', right_on='accession', how = 'outer')\n",
    "combine_df = combine_df[['PublishedName', 'NCBIname', 'Strains', 'accession', 'strain', 'Authority', 'taxid', 'DOI', 'URL' ]]\n",
    "#combine_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "8d07fbae",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style type=\"text/css\">\n",
       "#T_23618_row0_col0, #T_23618_row0_col1, #T_23618_row1_col0, #T_23618_row1_col1, #T_23618_row2_col0, #T_23618_row2_col1, #T_23618_row3_col0, #T_23618_row3_col1, #T_23618_row4_col0, #T_23618_row4_col1, #T_23618_row5_col0, #T_23618_row5_col1, #T_23618_row6_col0, #T_23618_row6_col1, #T_23618_row7_col0, #T_23618_row7_col1, #T_23618_row8_col0, #T_23618_row8_col1, #T_23618_row9_col0, #T_23618_row9_col1, #T_23618_row10_col0, #T_23618_row10_col1, #T_23618_row11_col0, #T_23618_row11_col1, #T_23618_row12_col0, #T_23618_row12_col1, #T_23618_row13_col0, #T_23618_row13_col1, #T_23618_row14_col0, #T_23618_row14_col1, #T_23618_row15_col0, #T_23618_row15_col1, #T_23618_row16_col0, #T_23618_row16_col1, #T_23618_row17_col0, #T_23618_row17_col1, #T_23618_row18_col0, #T_23618_row18_col1, #T_23618_row19_col0, #T_23618_row19_col1, #T_23618_row20_col0, #T_23618_row20_col1, #T_23618_row21_col0, #T_23618_row21_col1, #T_23618_row22_col0, #T_23618_row22_col1, #T_23618_row23_col0, #T_23618_row23_col1, #T_23618_row24_col0, #T_23618_row24_col1, #T_23618_row25_col0, #T_23618_row25_col1, #T_23618_row26_col0, #T_23618_row26_col1, #T_23618_row27_col0, #T_23618_row27_col1, #T_23618_row28_col0, #T_23618_row28_col1, #T_23618_row29_col0, #T_23618_row29_col1, #T_23618_row30_col0, #T_23618_row30_col1, #T_23618_row31_col0, #T_23618_row31_col1, #T_23618_row32_col0, #T_23618_row32_col1, #T_23618_row33_col0, #T_23618_row33_col1, #T_23618_row34_col0, #T_23618_row34_col1, #T_23618_row35_col0, #T_23618_row35_col1, #T_23618_row36_col0, #T_23618_row36_col1, #T_23618_row37_col0, #T_23618_row37_col1, #T_23618_row38_col0, #T_23618_row38_col1, #T_23618_row39_col0, #T_23618_row39_col1, #T_23618_row40_col0, #T_23618_row40_col1, #T_23618_row41_col0, #T_23618_row41_col1, #T_23618_row42_col0, #T_23618_row42_col1, #T_23618_row43_col0, #T_23618_row43_col1, #T_23618_row44_col0, #T_23618_row44_col1, #T_23618_row45_col0, #T_23618_row45_col1, #T_23618_row46_col0, #T_23618_row46_col1, #T_23618_row47_col0, #T_23618_row47_col1, #T_23618_row48_col0, #T_23618_row48_col1, #T_23618_row49_col0, #T_23618_row49_col1, #T_23618_row50_col0, #T_23618_row50_col1, #T_23618_row51_col0, #T_23618_row51_col1 {\n",
       "  background-color: #FFB3BA;\n",
       "}\n",
       "</style>\n",
       "<table id=\"T_23618\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th class=\"blank level0\" >&nbsp;</th>\n",
       "      <th id=\"T_23618_level0_col0\" class=\"col_heading level0 col0\" >PublishedName</th>\n",
       "      <th id=\"T_23618_level0_col1\" class=\"col_heading level0 col1\" >NCBIname</th>\n",
       "      <th id=\"T_23618_level0_col2\" class=\"col_heading level0 col2\" >Strains</th>\n",
       "      <th id=\"T_23618_level0_col3\" class=\"col_heading level0 col3\" >accession</th>\n",
       "      <th id=\"T_23618_level0_col4\" class=\"col_heading level0 col4\" >strain</th>\n",
       "      <th id=\"T_23618_level0_col5\" class=\"col_heading level0 col5\" >Authority</th>\n",
       "      <th id=\"T_23618_level0_col6\" class=\"col_heading level0 col6\" >taxid</th>\n",
       "      <th id=\"T_23618_level0_col7\" class=\"col_heading level0 col7\" >DOI</th>\n",
       "      <th id=\"T_23618_level0_col8\" class=\"col_heading level0 col8\" >URL</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row0\" class=\"row_heading level0 row0\" >0</th>\n",
       "      <td id=\"T_23618_row0_col0\" class=\"data row0 col0\" >Roseomonas cutis </td>\n",
       "      <td id=\"T_23618_row0_col1\" class=\"data row0 col1\" >Roseomonas sp. OT10</td>\n",
       "      <td id=\"T_23618_row0_col2\" class=\"data row0 col2\" > is OT10T (=KCTC 92087T =JCM 34968T, is 72.6 mol% (genome</td>\n",
       "      <td id=\"T_23618_row0_col3\" class=\"data row0 col3\" >OP788068</td>\n",
       "      <td id=\"T_23618_row0_col4\" class=\"data row0 col4\" >OT10</td>\n",
       "      <td id=\"T_23618_row0_col5\" class=\"data row0 col5\" >Son et al. 2025</td>\n",
       "      <td id=\"T_23618_row0_col6\" class=\"data row0 col6\" >2897332.000000</td>\n",
       "      <td id=\"T_23618_row0_col7\" class=\"data row0 col7\" >https://doi.org/10.1099/ijsem.0.006617</td>\n",
       "      <td id=\"T_23618_row0_col8\" class=\"data row0 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006617%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018546452%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=qPgbRuHApRO85zY1%2FXyHK7fwpRhZIQDtoO%2BMyTwMQ8I%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row1\" class=\"row_heading level0 row1\" >1</th>\n",
       "      <td id=\"T_23618_row1_col0\" class=\"data row1 col0\" >and Baghela </td>\n",
       "      <td id=\"T_23618_row1_col1\" class=\"data row1 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row1_col2\" class=\"data row1 col2\" ></td>\n",
       "      <td id=\"T_23618_row1_col3\" class=\"data row1 col3\" >MB855459</td>\n",
       "      <td id=\"T_23618_row1_col4\" class=\"data row1 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row1_col5\" class=\"data row1 col5\" >Sipiczki and Baghela 2025</td>\n",
       "      <td id=\"T_23618_row1_col6\" class=\"data row1 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row1_col7\" class=\"data row1 col7\" >https://doi.org/10.1099/ijsem.0.006629</td>\n",
       "      <td id=\"T_23618_row1_col8\" class=\"data row1 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006629%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018597448%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=gTvoT9taoY5EywTBMznivistMD3RQuxyiE0TB92Vbu8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row2\" class=\"row_heading level0 row2\" >2</th>\n",
       "      <td id=\"T_23618_row2_col0\" class=\"data row2 col0\" >Methylobacterium litchii </td>\n",
       "      <td id=\"T_23618_row2_col1\" class=\"data row2 col1\" >Methylobacterium sp.</td>\n",
       "      <td id=\"T_23618_row2_col2\" class=\"data row2 col2\" > is SD21T (=GDMCC 1.4327T=KCTC 8300T</td>\n",
       "      <td id=\"T_23618_row2_col3\" class=\"data row2 col3\" >PP600109</td>\n",
       "      <td id=\"T_23618_row2_col4\" class=\"data row2 col4\" >SD21</td>\n",
       "      <td id=\"T_23618_row2_col5\" class=\"data row2 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row2_col6\" class=\"data row2 col6\" >409.000000</td>\n",
       "      <td id=\"T_23618_row2_col7\" class=\"data row2 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row2_col8\" class=\"data row2 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row3\" class=\"row_heading level0 row3\" >3</th>\n",
       "      <td id=\"T_23618_row3_col0\" class=\"data row3 col0\" >Methylobacterium litchii </td>\n",
       "      <td id=\"T_23618_row3_col1\" class=\"data row3 col1\" >Methylobacterium sp. SD21</td>\n",
       "      <td id=\"T_23618_row3_col2\" class=\"data row3 col2\" > is SD21T (=GDMCC 1.4327T=KCTC 8300T</td>\n",
       "      <td id=\"T_23618_row3_col3\" class=\"data row3 col3\" >JBBXMY000000000</td>\n",
       "      <td id=\"T_23618_row3_col4\" class=\"data row3 col4\" >SD21</td>\n",
       "      <td id=\"T_23618_row3_col5\" class=\"data row3 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row3_col6\" class=\"data row3 col6\" >3138810.000000</td>\n",
       "      <td id=\"T_23618_row3_col7\" class=\"data row3 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row3_col8\" class=\"data row3 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row4\" class=\"row_heading level0 row4\" >4</th>\n",
       "      <td id=\"T_23618_row4_col0\" class=\"data row4 col0\" >Methylobacterium guangdongense </td>\n",
       "      <td id=\"T_23618_row4_col1\" class=\"data row4 col1\" >Methylobacterium sp.</td>\n",
       "      <td id=\"T_23618_row4_col2\" class=\"data row4 col2\" > is SI9T (=GDMCC 1.14329T= KCTC 8298T</td>\n",
       "      <td id=\"T_23618_row4_col3\" class=\"data row4 col3\" >PP600110</td>\n",
       "      <td id=\"T_23618_row4_col4\" class=\"data row4 col4\" >SI9</td>\n",
       "      <td id=\"T_23618_row4_col5\" class=\"data row4 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row4_col6\" class=\"data row4 col6\" >409.000000</td>\n",
       "      <td id=\"T_23618_row4_col7\" class=\"data row4 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row4_col8\" class=\"data row4 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row5\" class=\"row_heading level0 row5\" >5</th>\n",
       "      <td id=\"T_23618_row5_col0\" class=\"data row5 col0\" >Methylobacterium guangdongense </td>\n",
       "      <td id=\"T_23618_row5_col1\" class=\"data row5 col1\" >Methylobacterium sp. SI9</td>\n",
       "      <td id=\"T_23618_row5_col2\" class=\"data row5 col2\" > is SI9T (=GDMCC 1.14329T= KCTC 8298T</td>\n",
       "      <td id=\"T_23618_row5_col3\" class=\"data row5 col3\" >JBBXMX000000000</td>\n",
       "      <td id=\"T_23618_row5_col4\" class=\"data row5 col4\" >SI9</td>\n",
       "      <td id=\"T_23618_row5_col5\" class=\"data row5 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row5_col6\" class=\"data row5 col6\" >3138811.000000</td>\n",
       "      <td id=\"T_23618_row5_col7\" class=\"data row5 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row5_col8\" class=\"data row5 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row6\" class=\"row_heading level0 row6\" >6</th>\n",
       "      <td id=\"T_23618_row6_col0\" class=\"data row6 col0\" >Methylorubrum subtropicum </td>\n",
       "      <td id=\"T_23618_row6_col1\" class=\"data row6 col1\" >Methylorubrum sp.</td>\n",
       "      <td id=\"T_23618_row6_col2\" class=\"data row6 col2\" > is SB2T (=GDMCC 1.14328T= KCTC 8299T</td>\n",
       "      <td id=\"T_23618_row6_col3\" class=\"data row6 col3\" >PP600111</td>\n",
       "      <td id=\"T_23618_row6_col4\" class=\"data row6 col4\" >SB2</td>\n",
       "      <td id=\"T_23618_row6_col5\" class=\"data row6 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row6_col6\" class=\"data row6 col6\" >2282524.000000</td>\n",
       "      <td id=\"T_23618_row6_col7\" class=\"data row6 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row6_col8\" class=\"data row6 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row7\" class=\"row_heading level0 row7\" >7</th>\n",
       "      <td id=\"T_23618_row7_col0\" class=\"data row7 col0\" >Methylorubrum subtropicum </td>\n",
       "      <td id=\"T_23618_row7_col1\" class=\"data row7 col1\" >Methylorubrum sp. SB2</td>\n",
       "      <td id=\"T_23618_row7_col2\" class=\"data row7 col2\" > is SB2T (=GDMCC 1.14328T= KCTC 8299T</td>\n",
       "      <td id=\"T_23618_row7_col3\" class=\"data row7 col3\" >JBBXMW000000000</td>\n",
       "      <td id=\"T_23618_row7_col4\" class=\"data row7 col4\" >SB2</td>\n",
       "      <td id=\"T_23618_row7_col5\" class=\"data row7 col5\" >Gao et al. 2025</td>\n",
       "      <td id=\"T_23618_row7_col6\" class=\"data row7 col6\" >3138812.000000</td>\n",
       "      <td id=\"T_23618_row7_col7\" class=\"data row7 col7\" >https://doi.org/10.1099/ijsem.0.006639</td>\n",
       "      <td id=\"T_23618_row7_col8\" class=\"data row7 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006639%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018647513%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=EESf0RJ1F9zstzVMMd%2BGUCGXgkZBUotBItdAa2k1%2BA8%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row8\" class=\"row_heading level0 row8\" >8</th>\n",
       "      <td id=\"T_23618_row8_col0\" class=\"data row8 col0\" >Pseudomonas fungipugnans </td>\n",
       "      <td id=\"T_23618_row8_col1\" class=\"data row8 col1\" >Pseudomonas sp.</td>\n",
       "      <td id=\"T_23618_row8_col2\" class=\"data row8 col2\" ></td>\n",
       "      <td id=\"T_23618_row8_col3\" class=\"data row8 col3\" >OQ547267</td>\n",
       "      <td id=\"T_23618_row8_col4\" class=\"data row8 col4\" >#681</td>\n",
       "      <td id=\"T_23618_row8_col5\" class=\"data row8 col5\" >Schnyder et al. 2025</td>\n",
       "      <td id=\"T_23618_row8_col6\" class=\"data row8 col6\" >306.000000</td>\n",
       "      <td id=\"T_23618_row8_col7\" class=\"data row8 col7\" >https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td id=\"T_23618_row8_col8\" class=\"data row8 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row9\" class=\"row_heading level0 row9\" >9</th>\n",
       "      <td id=\"T_23618_row9_col0\" class=\"data row9 col0\" >Pseudomonas fungipugnans </td>\n",
       "      <td id=\"T_23618_row9_col1\" class=\"data row9 col1\" >Pseudomonas sp. 681</td>\n",
       "      <td id=\"T_23618_row9_col2\" class=\"data row9 col2\" ></td>\n",
       "      <td id=\"T_23618_row9_col3\" class=\"data row9 col3\" >JARBWL000000000</td>\n",
       "      <td id=\"T_23618_row9_col4\" class=\"data row9 col4\" >681</td>\n",
       "      <td id=\"T_23618_row9_col5\" class=\"data row9 col5\" >Schnyder et al. 2025</td>\n",
       "      <td id=\"T_23618_row9_col6\" class=\"data row9 col6\" >3024217.000000</td>\n",
       "      <td id=\"T_23618_row9_col7\" class=\"data row9 col7\" >https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td id=\"T_23618_row9_col8\" class=\"data row9 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row10\" class=\"row_heading level0 row10\" >10</th>\n",
       "      <td id=\"T_23618_row10_col0\" class=\"data row10 col0\" >Pseudomonas fungipugnans </td>\n",
       "      <td id=\"T_23618_row10_col1\" class=\"data row10 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row10_col2\" class=\"data row10 col2\" ></td>\n",
       "      <td id=\"T_23618_row10_col3\" class=\"data row10 col3\" >nan</td>\n",
       "      <td id=\"T_23618_row10_col4\" class=\"data row10 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row10_col5\" class=\"data row10 col5\" >Schnyder et al. 2025</td>\n",
       "      <td id=\"T_23618_row10_col6\" class=\"data row10 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row10_col7\" class=\"data row10 col7\" >https://doi.org/10.1099/ijsem.0.006624</td>\n",
       "      <td id=\"T_23618_row10_col8\" class=\"data row10 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006624%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018675073%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=%2FgfJBV%2BzhTm8J8IFPtJLLRJT15dKr4qhPOWJ5ISjMQY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row11\" class=\"row_heading level0 row11\" >11</th>\n",
       "      <td id=\"T_23618_row11_col0\" class=\"data row11 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row11_col1\" class=\"data row11 col1\" >Leptospira sp. WS58.C1</td>\n",
       "      <td id=\"T_23618_row11_col2\" class=\"data row11 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row11_col3\" class=\"data row11 col3\" >PQ066384</td>\n",
       "      <td id=\"T_23618_row11_col4\" class=\"data row11 col4\" >WS58.C1</td>\n",
       "      <td id=\"T_23618_row11_col5\" class=\"data row11 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row11_col6\" class=\"data row11 col6\" >3235173.000000</td>\n",
       "      <td id=\"T_23618_row11_col7\" class=\"data row11 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row11_col8\" class=\"data row11 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row12\" class=\"row_heading level0 row12\" >12</th>\n",
       "      <td id=\"T_23618_row12_col0\" class=\"data row12 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row12_col1\" class=\"data row12 col1\" >Leptospira sp. WS92.C1</td>\n",
       "      <td id=\"T_23618_row12_col2\" class=\"data row12 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row12_col3\" class=\"data row12 col3\" >PQ066385</td>\n",
       "      <td id=\"T_23618_row12_col4\" class=\"data row12 col4\" >WS92.C1</td>\n",
       "      <td id=\"T_23618_row12_col5\" class=\"data row12 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row12_col6\" class=\"data row12 col6\" >3235175.000000</td>\n",
       "      <td id=\"T_23618_row12_col7\" class=\"data row12 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row12_col8\" class=\"data row12 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row13\" class=\"row_heading level0 row13\" >13</th>\n",
       "      <td id=\"T_23618_row13_col0\" class=\"data row13 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row13_col1\" class=\"data row13 col1\" >Leptospira sp. WS39.C2</td>\n",
       "      <td id=\"T_23618_row13_col2\" class=\"data row13 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row13_col3\" class=\"data row13 col3\" >PQ066386</td>\n",
       "      <td id=\"T_23618_row13_col4\" class=\"data row13 col4\" >WS39.C2</td>\n",
       "      <td id=\"T_23618_row13_col5\" class=\"data row13 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row13_col6\" class=\"data row13 col6\" >3235171.000000</td>\n",
       "      <td id=\"T_23618_row13_col7\" class=\"data row13 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row13_col8\" class=\"data row13 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row14\" class=\"row_heading level0 row14\" >14</th>\n",
       "      <td id=\"T_23618_row14_col0\" class=\"data row14 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row14_col1\" class=\"data row14 col1\" >Leptospira sp. WS4.C2</td>\n",
       "      <td id=\"T_23618_row14_col2\" class=\"data row14 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row14_col3\" class=\"data row14 col3\" >PQ066387</td>\n",
       "      <td id=\"T_23618_row14_col4\" class=\"data row14 col4\" >WS4.C2</td>\n",
       "      <td id=\"T_23618_row14_col5\" class=\"data row14 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row14_col6\" class=\"data row14 col6\" >3235172.000000</td>\n",
       "      <td id=\"T_23618_row14_col7\" class=\"data row14 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row14_col8\" class=\"data row14 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row15\" class=\"row_heading level0 row15\" >15</th>\n",
       "      <td id=\"T_23618_row15_col0\" class=\"data row15 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row15_col1\" class=\"data row15 col1\" >Leptospira sp. WS60.C2</td>\n",
       "      <td id=\"T_23618_row15_col2\" class=\"data row15 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row15_col3\" class=\"data row15 col3\" >PQ066388</td>\n",
       "      <td id=\"T_23618_row15_col4\" class=\"data row15 col4\" >WS60.C2</td>\n",
       "      <td id=\"T_23618_row15_col5\" class=\"data row15 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row15_col6\" class=\"data row15 col6\" >3235174.000000</td>\n",
       "      <td id=\"T_23618_row15_col7\" class=\"data row15 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row15_col8\" class=\"data row15 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row16\" class=\"row_heading level0 row16\" >16</th>\n",
       "      <td id=\"T_23618_row16_col0\" class=\"data row16 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row16_col1\" class=\"data row16 col1\" >Leptospira sp. WS58.C1</td>\n",
       "      <td id=\"T_23618_row16_col2\" class=\"data row16 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row16_col3\" class=\"data row16 col3\" >CP162137</td>\n",
       "      <td id=\"T_23618_row16_col4\" class=\"data row16 col4\" >WS58.C1</td>\n",
       "      <td id=\"T_23618_row16_col5\" class=\"data row16 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row16_col6\" class=\"data row16 col6\" >3235173.000000</td>\n",
       "      <td id=\"T_23618_row16_col7\" class=\"data row16 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row16_col8\" class=\"data row16 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row17\" class=\"row_heading level0 row17\" >17</th>\n",
       "      <td id=\"T_23618_row17_col0\" class=\"data row17 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row17_col1\" class=\"data row17 col1\" >Leptospira sp. WS58.C1</td>\n",
       "      <td id=\"T_23618_row17_col2\" class=\"data row17 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row17_col3\" class=\"data row17 col3\" >CP162138</td>\n",
       "      <td id=\"T_23618_row17_col4\" class=\"data row17 col4\" >WS58.C1</td>\n",
       "      <td id=\"T_23618_row17_col5\" class=\"data row17 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row17_col6\" class=\"data row17 col6\" >3235173.000000</td>\n",
       "      <td id=\"T_23618_row17_col7\" class=\"data row17 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row17_col8\" class=\"data row17 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row18\" class=\"row_heading level0 row18\" >18</th>\n",
       "      <td id=\"T_23618_row18_col0\" class=\"data row18 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row18_col1\" class=\"data row18 col1\" >Leptospira sp. WS92.C1</td>\n",
       "      <td id=\"T_23618_row18_col2\" class=\"data row18 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row18_col3\" class=\"data row18 col3\" >CP162130</td>\n",
       "      <td id=\"T_23618_row18_col4\" class=\"data row18 col4\" >WS92.C1</td>\n",
       "      <td id=\"T_23618_row18_col5\" class=\"data row18 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row18_col6\" class=\"data row18 col6\" >3235175.000000</td>\n",
       "      <td id=\"T_23618_row18_col7\" class=\"data row18 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row18_col8\" class=\"data row18 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row19\" class=\"row_heading level0 row19\" >19</th>\n",
       "      <td id=\"T_23618_row19_col0\" class=\"data row19 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row19_col1\" class=\"data row19 col1\" >Leptospira sp. WS92.C1</td>\n",
       "      <td id=\"T_23618_row19_col2\" class=\"data row19 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row19_col3\" class=\"data row19 col3\" >CP162132</td>\n",
       "      <td id=\"T_23618_row19_col4\" class=\"data row19 col4\" >WS92.C1</td>\n",
       "      <td id=\"T_23618_row19_col5\" class=\"data row19 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row19_col6\" class=\"data row19 col6\" >3235175.000000</td>\n",
       "      <td id=\"T_23618_row19_col7\" class=\"data row19 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row19_col8\" class=\"data row19 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row20\" class=\"row_heading level0 row20\" >20</th>\n",
       "      <td id=\"T_23618_row20_col0\" class=\"data row20 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row20_col1\" class=\"data row20 col1\" >Leptospira sp. WS39.C2</td>\n",
       "      <td id=\"T_23618_row20_col2\" class=\"data row20 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row20_col3\" class=\"data row20 col3\" >CP162142</td>\n",
       "      <td id=\"T_23618_row20_col4\" class=\"data row20 col4\" >WS39.C2</td>\n",
       "      <td id=\"T_23618_row20_col5\" class=\"data row20 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row20_col6\" class=\"data row20 col6\" >3235171.000000</td>\n",
       "      <td id=\"T_23618_row20_col7\" class=\"data row20 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row20_col8\" class=\"data row20 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row21\" class=\"row_heading level0 row21\" >21</th>\n",
       "      <td id=\"T_23618_row21_col0\" class=\"data row21 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row21_col1\" class=\"data row21 col1\" >Leptospira sp. WS39.C2</td>\n",
       "      <td id=\"T_23618_row21_col2\" class=\"data row21 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row21_col3\" class=\"data row21 col3\" >CP162144</td>\n",
       "      <td id=\"T_23618_row21_col4\" class=\"data row21 col4\" >WS39.C2</td>\n",
       "      <td id=\"T_23618_row21_col5\" class=\"data row21 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row21_col6\" class=\"data row21 col6\" >3235171.000000</td>\n",
       "      <td id=\"T_23618_row21_col7\" class=\"data row21 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row21_col8\" class=\"data row21 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row22\" class=\"row_heading level0 row22\" >22</th>\n",
       "      <td id=\"T_23618_row22_col0\" class=\"data row22 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row22_col1\" class=\"data row22 col1\" >Leptospira sp. WS4.C2</td>\n",
       "      <td id=\"T_23618_row22_col2\" class=\"data row22 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row22_col3\" class=\"data row22 col3\" >CP162139</td>\n",
       "      <td id=\"T_23618_row22_col4\" class=\"data row22 col4\" >WS4.C2</td>\n",
       "      <td id=\"T_23618_row22_col5\" class=\"data row22 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row22_col6\" class=\"data row22 col6\" >3235172.000000</td>\n",
       "      <td id=\"T_23618_row22_col7\" class=\"data row22 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row22_col8\" class=\"data row22 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row23\" class=\"row_heading level0 row23\" >23</th>\n",
       "      <td id=\"T_23618_row23_col0\" class=\"data row23 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row23_col1\" class=\"data row23 col1\" >Leptospira sp. WS4.C2</td>\n",
       "      <td id=\"T_23618_row23_col2\" class=\"data row23 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row23_col3\" class=\"data row23 col3\" >CP162141</td>\n",
       "      <td id=\"T_23618_row23_col4\" class=\"data row23 col4\" >WS4.C2</td>\n",
       "      <td id=\"T_23618_row23_col5\" class=\"data row23 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row23_col6\" class=\"data row23 col6\" >3235172.000000</td>\n",
       "      <td id=\"T_23618_row23_col7\" class=\"data row23 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row23_col8\" class=\"data row23 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row24\" class=\"row_heading level0 row24\" >24</th>\n",
       "      <td id=\"T_23618_row24_col0\" class=\"data row24 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row24_col1\" class=\"data row24 col1\" >Leptospira sp. WS60.C2</td>\n",
       "      <td id=\"T_23618_row24_col2\" class=\"data row24 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row24_col3\" class=\"data row24 col3\" >CP162133</td>\n",
       "      <td id=\"T_23618_row24_col4\" class=\"data row24 col4\" >WS60.C2</td>\n",
       "      <td id=\"T_23618_row24_col5\" class=\"data row24 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row24_col6\" class=\"data row24 col6\" >3235174.000000</td>\n",
       "      <td id=\"T_23618_row24_col7\" class=\"data row24 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row24_col8\" class=\"data row24 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row25\" class=\"row_heading level0 row25\" >25</th>\n",
       "      <td id=\"T_23618_row25_col0\" class=\"data row25 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row25_col1\" class=\"data row25 col1\" >Leptospira sp. WS60.C2</td>\n",
       "      <td id=\"T_23618_row25_col2\" class=\"data row25 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row25_col3\" class=\"data row25 col3\" >CP162136</td>\n",
       "      <td id=\"T_23618_row25_col4\" class=\"data row25 col4\" >WS60.C2</td>\n",
       "      <td id=\"T_23618_row25_col5\" class=\"data row25 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row25_col6\" class=\"data row25 col6\" >3235174.000000</td>\n",
       "      <td id=\"T_23618_row25_col7\" class=\"data row25 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row25_col8\" class=\"data row25 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row26\" class=\"row_heading level0 row26\" >26</th>\n",
       "      <td id=\"T_23618_row26_col0\" class=\"data row26 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row26_col1\" class=\"data row26 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row26_col2\" class=\"data row26 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row26_col3\" class=\"data row26 col3\" >SAMN42533266</td>\n",
       "      <td id=\"T_23618_row26_col4\" class=\"data row26 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row26_col5\" class=\"data row26 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row26_col6\" class=\"data row26 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row26_col7\" class=\"data row26 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row26_col8\" class=\"data row26 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row27\" class=\"row_heading level0 row27\" >27</th>\n",
       "      <td id=\"T_23618_row27_col0\" class=\"data row27 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row27_col1\" class=\"data row27 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row27_col2\" class=\"data row27 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row27_col3\" class=\"data row27 col3\" >SAMN42533265</td>\n",
       "      <td id=\"T_23618_row27_col4\" class=\"data row27 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row27_col5\" class=\"data row27 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row27_col6\" class=\"data row27 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row27_col7\" class=\"data row27 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row27_col8\" class=\"data row27 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row28\" class=\"row_heading level0 row28\" >28</th>\n",
       "      <td id=\"T_23618_row28_col0\" class=\"data row28 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row28_col1\" class=\"data row28 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row28_col2\" class=\"data row28 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row28_col3\" class=\"data row28 col3\" >SAMN42533264</td>\n",
       "      <td id=\"T_23618_row28_col4\" class=\"data row28 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row28_col5\" class=\"data row28 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row28_col6\" class=\"data row28 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row28_col7\" class=\"data row28 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row28_col8\" class=\"data row28 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row29\" class=\"row_heading level0 row29\" >29</th>\n",
       "      <td id=\"T_23618_row29_col0\" class=\"data row29 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row29_col1\" class=\"data row29 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row29_col2\" class=\"data row29 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row29_col3\" class=\"data row29 col3\" >SAMN42533263</td>\n",
       "      <td id=\"T_23618_row29_col4\" class=\"data row29 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row29_col5\" class=\"data row29 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row29_col6\" class=\"data row29 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row29_col7\" class=\"data row29 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row29_col8\" class=\"data row29 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row30\" class=\"row_heading level0 row30\" >30</th>\n",
       "      <td id=\"T_23618_row30_col0\" class=\"data row30 col0\" >Leptospira gorisiae </td>\n",
       "      <td id=\"T_23618_row30_col1\" class=\"data row30 col1\" >nan</td>\n",
       "      <td id=\"T_23618_row30_col2\" class=\"data row30 col2\" > WS92.C1T (=NVSL-WS92.C1T=KIT0303T</td>\n",
       "      <td id=\"T_23618_row30_col3\" class=\"data row30 col3\" >SAMN42533262</td>\n",
       "      <td id=\"T_23618_row30_col4\" class=\"data row30 col4\" >nan</td>\n",
       "      <td id=\"T_23618_row30_col5\" class=\"data row30 col5\" >Hamond et al. 2025</td>\n",
       "      <td id=\"T_23618_row30_col6\" class=\"data row30 col6\" >nan</td>\n",
       "      <td id=\"T_23618_row30_col7\" class=\"data row30 col7\" >https://doi.org/10.1099/ijsem.0.006595</td>\n",
       "      <td id=\"T_23618_row30_col8\" class=\"data row30 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006595%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018724864%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=XAG3CvJH%2BgeCTKqe1Z%2BaHj6WUtpMAoVbkdJszMo%2BkdU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row31\" class=\"row_heading level0 row31\" >31</th>\n",
       "      <td id=\"T_23618_row31_col0\" class=\"data row31 col0\" >Luteimonas salinilitoris </td>\n",
       "      <td id=\"T_23618_row31_col1\" class=\"data row31 col1\" >Luteimonas sp. C3_2_a3</td>\n",
       "      <td id=\"T_23618_row31_col2\" class=\"data row31 col2\" > C3-2-a3T (CGMCC 1.14507T = KCTC 8642T</td>\n",
       "      <td id=\"T_23618_row31_col3\" class=\"data row31 col3\" >PQ065621</td>\n",
       "      <td id=\"T_23618_row31_col4\" class=\"data row31 col4\" >C3_2_a3</td>\n",
       "      <td id=\"T_23618_row31_col5\" class=\"data row31 col5\" >Zhang et al. 2025</td>\n",
       "      <td id=\"T_23618_row31_col6\" class=\"data row31 col6\" >3237697.000000</td>\n",
       "      <td id=\"T_23618_row31_col7\" class=\"data row31 col7\" >https://doi.org/10.1099/ijsem.0.006630</td>\n",
       "      <td id=\"T_23618_row31_col8\" class=\"data row31 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row32\" class=\"row_heading level0 row32\" >32</th>\n",
       "      <td id=\"T_23618_row32_col0\" class=\"data row32 col0\" >Luteimonas salinilitoris </td>\n",
       "      <td id=\"T_23618_row32_col1\" class=\"data row32 col1\" >Luteimonas sp.</td>\n",
       "      <td id=\"T_23618_row32_col2\" class=\"data row32 col2\" > C3-2-a3T (CGMCC 1.14507T = KCTC 8642T</td>\n",
       "      <td id=\"T_23618_row32_col3\" class=\"data row32 col3\" >PQ151652</td>\n",
       "      <td id=\"T_23618_row32_col4\" class=\"data row32 col4\" >B3_2_R+30</td>\n",
       "      <td id=\"T_23618_row32_col5\" class=\"data row32 col5\" >Zhang et al. 2025</td>\n",
       "      <td id=\"T_23618_row32_col6\" class=\"data row32 col6\" >1873125.000000</td>\n",
       "      <td id=\"T_23618_row32_col7\" class=\"data row32 col7\" >https://doi.org/10.1099/ijsem.0.006630</td>\n",
       "      <td id=\"T_23618_row32_col8\" class=\"data row32 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row33\" class=\"row_heading level0 row33\" >33</th>\n",
       "      <td id=\"T_23618_row33_col0\" class=\"data row33 col0\" >Luteimonas salinilitoris </td>\n",
       "      <td id=\"T_23618_row33_col1\" class=\"data row33 col1\" >Luteimonas sp. C3_2_a3</td>\n",
       "      <td id=\"T_23618_row33_col2\" class=\"data row33 col2\" > C3-2-a3T (CGMCC 1.14507T = KCTC 8642T</td>\n",
       "      <td id=\"T_23618_row33_col3\" class=\"data row33 col3\" >JBFWIB000000000</td>\n",
       "      <td id=\"T_23618_row33_col4\" class=\"data row33 col4\" >C3_2_a3</td>\n",
       "      <td id=\"T_23618_row33_col5\" class=\"data row33 col5\" >Zhang et al. 2025</td>\n",
       "      <td id=\"T_23618_row33_col6\" class=\"data row33 col6\" >3237697.000000</td>\n",
       "      <td id=\"T_23618_row33_col7\" class=\"data row33 col7\" >https://doi.org/10.1099/ijsem.0.006630</td>\n",
       "      <td id=\"T_23618_row33_col8\" class=\"data row33 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row34\" class=\"row_heading level0 row34\" >34</th>\n",
       "      <td id=\"T_23618_row34_col0\" class=\"data row34 col0\" >Luteimonas salinilitoris </td>\n",
       "      <td id=\"T_23618_row34_col1\" class=\"data row34 col1\" >Luteimonas sp. B3_2_R+30</td>\n",
       "      <td id=\"T_23618_row34_col2\" class=\"data row34 col2\" > C3-2-a3T (CGMCC 1.14507T = KCTC 8642T</td>\n",
       "      <td id=\"T_23618_row34_col3\" class=\"data row34 col3\" >JBFWIC000000000</td>\n",
       "      <td id=\"T_23618_row34_col4\" class=\"data row34 col4\" >B3_2_R+30</td>\n",
       "      <td id=\"T_23618_row34_col5\" class=\"data row34 col5\" >Zhang et al. 2025</td>\n",
       "      <td id=\"T_23618_row34_col6\" class=\"data row34 col6\" >3239301.000000</td>\n",
       "      <td id=\"T_23618_row34_col7\" class=\"data row34 col7\" >https://doi.org/10.1099/ijsem.0.006630</td>\n",
       "      <td id=\"T_23618_row34_col8\" class=\"data row34 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006630%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018801546%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Rnz2S9CR%2Br8nPHmvWNGcOpqTUmmZa5vHZDliCz87MLw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row35\" class=\"row_heading level0 row35\" >35</th>\n",
       "      <td id=\"T_23618_row35_col0\" class=\"data row35 col0\" >Dyella aluminiiresistens </td>\n",
       "      <td id=\"T_23618_row35_col1\" class=\"data row35 col1\" >Dyella sp.</td>\n",
       "      <td id=\"T_23618_row35_col2\" class=\"data row35 col2\" >, A6T (= GDMCC 1.4640T = KCTC 92542T</td>\n",
       "      <td id=\"T_23618_row35_col3\" class=\"data row35 col3\" >OR206507</td>\n",
       "      <td id=\"T_23618_row35_col4\" class=\"data row35 col4\" >A6</td>\n",
       "      <td id=\"T_23618_row35_col5\" class=\"data row35 col5\" >Li et al. 2025</td>\n",
       "      <td id=\"T_23618_row35_col6\" class=\"data row35 col6\" >1869338.000000</td>\n",
       "      <td id=\"T_23618_row35_col7\" class=\"data row35 col7\" >https://doi.org/10.1099/ijsem.0.006611</td>\n",
       "      <td id=\"T_23618_row35_col8\" class=\"data row35 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006611%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018853339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=S4R1T9Bs6FJngYjedz8T7GqhIOrzYpyrq3%2BkVjxy5%2Bw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row36\" class=\"row_heading level0 row36\" >36</th>\n",
       "      <td id=\"T_23618_row36_col0\" class=\"data row36 col0\" >Dyella aluminiiresistens </td>\n",
       "      <td id=\"T_23618_row36_col1\" class=\"data row36 col1\" >Dyella sp. A6</td>\n",
       "      <td id=\"T_23618_row36_col2\" class=\"data row36 col2\" >, A6T (= GDMCC 1.4640T = KCTC 92542T</td>\n",
       "      <td id=\"T_23618_row36_col3\" class=\"data row36 col3\" >CP132911</td>\n",
       "      <td id=\"T_23618_row36_col4\" class=\"data row36 col4\" >A6</td>\n",
       "      <td id=\"T_23618_row36_col5\" class=\"data row36 col5\" >Li et al. 2025</td>\n",
       "      <td id=\"T_23618_row36_col6\" class=\"data row36 col6\" >3069105.000000</td>\n",
       "      <td id=\"T_23618_row36_col7\" class=\"data row36 col7\" >https://doi.org/10.1099/ijsem.0.006611</td>\n",
       "      <td id=\"T_23618_row36_col8\" class=\"data row36 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006611%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018853339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=S4R1T9Bs6FJngYjedz8T7GqhIOrzYpyrq3%2BkVjxy5%2Bw%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row37\" class=\"row_heading level0 row37\" >37</th>\n",
       "      <td id=\"T_23618_row37_col0\" class=\"data row37 col0\" >Pseudonocardia spirodelae </td>\n",
       "      <td id=\"T_23618_row37_col1\" class=\"data row37 col1\" >Pseudonocardia sp. DW16-2</td>\n",
       "      <td id=\"T_23618_row37_col2\" class=\"data row37 col2\" > is DW16-2T (= TBRC 16418T = NBRC 115857T</td>\n",
       "      <td id=\"T_23618_row37_col3\" class=\"data row37 col3\" >JBBJUP000000000</td>\n",
       "      <td id=\"T_23618_row37_col4\" class=\"data row37 col4\" >DW16-2</td>\n",
       "      <td id=\"T_23618_row37_col5\" class=\"data row37 col5\" >Butdee et al. 2025</td>\n",
       "      <td id=\"T_23618_row37_col6\" class=\"data row37 col6\" >3133431.000000</td>\n",
       "      <td id=\"T_23618_row37_col7\" class=\"data row37 col7\" >https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td id=\"T_23618_row37_col8\" class=\"data row37 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row38\" class=\"row_heading level0 row38\" >38</th>\n",
       "      <td id=\"T_23618_row38_col0\" class=\"data row38 col0\" >Pseudonocardia spirodelae </td>\n",
       "      <td id=\"T_23618_row38_col1\" class=\"data row38 col1\" >Pseudonocardia sp.</td>\n",
       "      <td id=\"T_23618_row38_col2\" class=\"data row38 col2\" > is DW16-2T (= TBRC 16418T = NBRC 115857T</td>\n",
       "      <td id=\"T_23618_row38_col3\" class=\"data row38 col3\" >PP738009</td>\n",
       "      <td id=\"T_23618_row38_col4\" class=\"data row38 col4\" >DW16-2</td>\n",
       "      <td id=\"T_23618_row38_col5\" class=\"data row38 col5\" >Butdee et al. 2025</td>\n",
       "      <td id=\"T_23618_row38_col6\" class=\"data row38 col6\" >60912.000000</td>\n",
       "      <td id=\"T_23618_row38_col7\" class=\"data row38 col7\" >https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td id=\"T_23618_row38_col8\" class=\"data row38 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row39\" class=\"row_heading level0 row39\" >39</th>\n",
       "      <td id=\"T_23618_row39_col0\" class=\"data row39 col0\" >Pseudonocardia alni subsp. carboxydivorans </td>\n",
       "      <td id=\"T_23618_row39_col1\" class=\"data row39 col1\" >Pseudonocardia carboxydivorans</td>\n",
       "      <td id=\"T_23618_row39_col2\" class=\"data row39 col2\" > is JCM 14827T (= Y8T = SWP-2006T = KCCM 42678T = DSM 45349T</td>\n",
       "      <td id=\"T_23618_row39_col3\" class=\"data row39 col3\" >JBBPIX000000000</td>\n",
       "      <td id=\"T_23618_row39_col4\" class=\"data row39 col4\" >JCM 14827</td>\n",
       "      <td id=\"T_23618_row39_col5\" class=\"data row39 col5\" >Butdee et al. 2025</td>\n",
       "      <td id=\"T_23618_row39_col6\" class=\"data row39 col6\" >415010.000000</td>\n",
       "      <td id=\"T_23618_row39_col7\" class=\"data row39 col7\" >https://doi.org/10.1099/ijsem.0.006608</td>\n",
       "      <td id=\"T_23618_row39_col8\" class=\"data row39 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006608%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045018952382%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=lGG%2FxQai%2B2BoytjlA4v5JrDgks%2BiS7nfrhRzI%2FcqTWY%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row40\" class=\"row_heading level0 row40\" >40</th>\n",
       "      <td id=\"T_23618_row40_col0\" class=\"data row40 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row40_col1\" class=\"data row40 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row40_col2\" class=\"data row40 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row40_col3\" class=\"data row40 col3\" >LC786857</td>\n",
       "      <td id=\"T_23618_row40_col4\" class=\"data row40 col4\" >DMKU-SG26</td>\n",
       "      <td id=\"T_23618_row40_col5\" class=\"data row40 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row40_col6\" class=\"data row40 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row40_col7\" class=\"data row40 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row40_col8\" class=\"data row40 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row41\" class=\"row_heading level0 row41\" >41</th>\n",
       "      <td id=\"T_23618_row41_col0\" class=\"data row41 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row41_col1\" class=\"data row41 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row41_col2\" class=\"data row41 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row41_col3\" class=\"data row41 col3\" >LC832838</td>\n",
       "      <td id=\"T_23618_row41_col4\" class=\"data row41 col4\" >DMKU-RGM25</td>\n",
       "      <td id=\"T_23618_row41_col5\" class=\"data row41 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row41_col6\" class=\"data row41 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row41_col7\" class=\"data row41 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row41_col8\" class=\"data row41 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row42\" class=\"row_heading level0 row42\" >42</th>\n",
       "      <td id=\"T_23618_row42_col0\" class=\"data row42 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row42_col1\" class=\"data row42 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row42_col2\" class=\"data row42 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row42_col3\" class=\"data row42 col3\" >LC832837</td>\n",
       "      <td id=\"T_23618_row42_col4\" class=\"data row42 col4\" >DMKU-SG42</td>\n",
       "      <td id=\"T_23618_row42_col5\" class=\"data row42 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row42_col6\" class=\"data row42 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row42_col7\" class=\"data row42 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row42_col8\" class=\"data row42 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row43\" class=\"row_heading level0 row43\" >43</th>\n",
       "      <td id=\"T_23618_row43_col0\" class=\"data row43 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row43_col1\" class=\"data row43 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row43_col2\" class=\"data row43 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row43_col3\" class=\"data row43 col3\" >LC832840</td>\n",
       "      <td id=\"T_23618_row43_col4\" class=\"data row43 col4\" >DMKU-SYM22</td>\n",
       "      <td id=\"T_23618_row43_col5\" class=\"data row43 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row43_col6\" class=\"data row43 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row43_col7\" class=\"data row43 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row43_col8\" class=\"data row43 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row44\" class=\"row_heading level0 row44\" >44</th>\n",
       "      <td id=\"T_23618_row44_col0\" class=\"data row44 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row44_col1\" class=\"data row44 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row44_col2\" class=\"data row44 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row44_col3\" class=\"data row44 col3\" >LC832839</td>\n",
       "      <td id=\"T_23618_row44_col4\" class=\"data row44 col4\" >DMKU-RG41</td>\n",
       "      <td id=\"T_23618_row44_col5\" class=\"data row44 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row44_col6\" class=\"data row44 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row44_col7\" class=\"data row44 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row44_col8\" class=\"data row44 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row45\" class=\"row_heading level0 row45\" >45</th>\n",
       "      <td id=\"T_23618_row45_col0\" class=\"data row45 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row45_col1\" class=\"data row45 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row45_col2\" class=\"data row45 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row45_col3\" class=\"data row45 col3\" >LC818861</td>\n",
       "      <td id=\"T_23618_row45_col4\" class=\"data row45 col4\" >DMKU-RX317</td>\n",
       "      <td id=\"T_23618_row45_col5\" class=\"data row45 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row45_col6\" class=\"data row45 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row45_col7\" class=\"data row45 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row45_col8\" class=\"data row45 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row46\" class=\"row_heading level0 row46\" >46</th>\n",
       "      <td id=\"T_23618_row46_col0\" class=\"data row46 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row46_col1\" class=\"data row46 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row46_col2\" class=\"data row46 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row46_col3\" class=\"data row46 col3\" >LC797015</td>\n",
       "      <td id=\"T_23618_row46_col4\" class=\"data row46 col4\" >DMKU-SG26</td>\n",
       "      <td id=\"T_23618_row46_col5\" class=\"data row46 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row46_col6\" class=\"data row46 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row46_col7\" class=\"data row46 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row46_col8\" class=\"data row46 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row47\" class=\"row_heading level0 row47\" >47</th>\n",
       "      <td id=\"T_23618_row47_col0\" class=\"data row47 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row47_col1\" class=\"data row47 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row47_col2\" class=\"data row47 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row47_col3\" class=\"data row47 col3\" >LC832841</td>\n",
       "      <td id=\"T_23618_row47_col4\" class=\"data row47 col4\" >DMKU-RGM25</td>\n",
       "      <td id=\"T_23618_row47_col5\" class=\"data row47 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row47_col6\" class=\"data row47 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row47_col7\" class=\"data row47 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row47_col8\" class=\"data row47 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row48\" class=\"row_heading level0 row48\" >48</th>\n",
       "      <td id=\"T_23618_row48_col0\" class=\"data row48 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row48_col1\" class=\"data row48 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row48_col2\" class=\"data row48 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row48_col3\" class=\"data row48 col3\" >LC797016</td>\n",
       "      <td id=\"T_23618_row48_col4\" class=\"data row48 col4\" >DMKU-SG42</td>\n",
       "      <td id=\"T_23618_row48_col5\" class=\"data row48 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row48_col6\" class=\"data row48 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row48_col7\" class=\"data row48 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row48_col8\" class=\"data row48 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row49\" class=\"row_heading level0 row49\" >49</th>\n",
       "      <td id=\"T_23618_row49_col0\" class=\"data row49 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row49_col1\" class=\"data row49 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row49_col2\" class=\"data row49 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row49_col3\" class=\"data row49 col3\" >LC832842</td>\n",
       "      <td id=\"T_23618_row49_col4\" class=\"data row49 col4\" >DMKU-SYM22</td>\n",
       "      <td id=\"T_23618_row49_col5\" class=\"data row49 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row49_col6\" class=\"data row49 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row49_col7\" class=\"data row49 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row49_col8\" class=\"data row49 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row50\" class=\"row_heading level0 row50\" >50</th>\n",
       "      <td id=\"T_23618_row50_col0\" class=\"data row50 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row50_col1\" class=\"data row50 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row50_col2\" class=\"data row50 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row50_col3\" class=\"data row50 col3\" >LC832843</td>\n",
       "      <td id=\"T_23618_row50_col4\" class=\"data row50 col4\" >DMKU-RG41</td>\n",
       "      <td id=\"T_23618_row50_col5\" class=\"data row50 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row50_col6\" class=\"data row50 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row50_col7\" class=\"data row50 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row50_col8\" class=\"data row50 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th id=\"T_23618_level0_row51\" class=\"row_heading level0 row51\" >51</th>\n",
       "      <td id=\"T_23618_row51_col0\" class=\"data row51 col0\" >and Khunnamwong </td>\n",
       "      <td id=\"T_23618_row51_col1\" class=\"data row51 col1\" >Vishniacozyma sp.</td>\n",
       "      <td id=\"T_23618_row51_col2\" class=\"data row51 col2\" > was isolated</td>\n",
       "      <td id=\"T_23618_row51_col3\" class=\"data row51 col3\" >LC832855</td>\n",
       "      <td id=\"T_23618_row51_col4\" class=\"data row51 col4\" >DMKU-RX317</td>\n",
       "      <td id=\"T_23618_row51_col5\" class=\"data row51 col5\" >Gungprakhon et al. 2025</td>\n",
       "      <td id=\"T_23618_row51_col6\" class=\"data row51 col6\" >1902918.000000</td>\n",
       "      <td id=\"T_23618_row51_col7\" class=\"data row51 col7\" >https://doi.org/10.1099/ijsem.0.006623</td>\n",
       "      <td id=\"T_23618_row51_col8\" class=\"data row51 col8\" >https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fijsem%2F10.1099%2Fijsem.0.006623%3Femailalert%3Dtrue&data=05%7C02%7Cmcveigh%40ncbi.nlm.nih.gov%7C7d22bd5abc544b8c9a8308dd3167fac7%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638721045019003789%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C60000%7C%7C%7C&sdata=Y0l69VL7MULx%2Bf5hP3OL4pKnGnSeAyEfRX41xAWkGZU%3D&reserved=0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n"
      ],
      "text/plain": [
       "<pandas.io.formats.style.Styler at 0x7f92009a4820>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def highlight_rows(row):\n",
    "    ijsemvalue = row.loc['PublishedName']\n",
    "    ncbivalue = row.loc['NCBIname']\n",
    "    if ijsemvalue != ncbivalue:\n",
    "        color = '#FFB3BA' # Red\n",
    "    elif ijsemvalue == ncbivalue:\n",
    "        color = '#BAFFC9' # Green\n",
    "    return ['background-color: {}'.format(color) for r in row]\n",
    "\n",
    "new_df = combine_df.style.apply(highlight_rows, axis=1, subset=['PublishedName', 'NCBIname'])\n",
    "new_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5aa05366",
   "metadata": {},
   "source": [
    "### write output to excel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "fb8948c8",
   "metadata": {},
   "outputs": [],
   "source": [
    "new_df.to_excel(output, engine='xlsxwriter', index = False, na_rep = '') "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3ca925cd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "652c0d09",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "ec74636d",
   "metadata": {},
   "source": [
    "### taxonerd test -- skip this"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8d865873",
   "metadata": {},
   "outputs": [],
   "source": [
    "from taxonerd import TaxoNERD\n",
    "taxonerd = TaxoNERD(prefer_gpu=False)\n",
    "nlp = taxonerd.load(model=\"en_ner_eco_md\", exclude=[], linker=\"taxref\", threshold=0.7)\n",
    "nlp.pipe_names"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f0783a12",
   "metadata": {},
   "source": [
    "### option to use biopython to retrieve orgnames based on accessions -- this approarch works but using srcchk for now so this isn't needed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d3e319d0",
   "metadata": {},
   "outputs": [],
   "source": [
    "import Bio\n",
    "from Bio import Entrez\n",
    "from Bio import SeqIO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "12912624",
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(db=\"nucleotide\", id=\"AY851612\", rettype=\"gb\", retmode=\"text\")\n",
    "x = SeqIO.read(handle, 'genbank')\n",
    "x.annotations['organism']"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "59d63926",
   "metadata": {},
   "source": [
    "### extra stuff not using, just saving for now"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7380ffa1",
   "metadata": {},
   "outputs": [],
   "source": [
    "combine_df['boo'] = combine_df['PublishedName'] != combine_df['NCBIname']\n",
    "\n",
    "combine_df.style.applymap('background-color: red', subset=['PublishedName'])\n",
    "\n",
    "combine_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7591cab8",
   "metadata": {},
   "outputs": [],
   "source": [
    " def color_positive_green(val):\n",
    "    if combine_df.iloc['boo'] == True:\n",
    "        color = 'red'\n",
    "    else:\n",
    "        color = 'black'\n",
    "    return 'color: %s' % color\n",
    " \n",
    "combine_df.style.applymap(color_positive_green)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "88ffd0f6",
   "metadata": {},
   "outputs": [],
   "source": [
    "#combine_df.apply_style_by_indexes(combine_df[combine_df['PublishedName'] != combine_df['NCBIname']], styler_obj=Styler(bg_color='red'), cols_to_style=['PublishedName', 'NCBIname'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "08a7f93d",
   "metadata": {},
   "outputs": [],
   "source": [
    "#df.style.apply(color, axis=None).to_excel('styled.xlsx', engine='openpyxl')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e57e3a7f",
   "metadata": {},
   "source": [
    "### Selenium web scraper - title and author note (footnote 1), some of these functions can be turned off following additional testing. Currently not using this function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fe545449",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Function to scrape headlines using Selenium\n",
    "def scrape_with_selenium(url, counter, doi):\n",
    "    global title\n",
    "    global footnote\n",
    "    global description\n",
    "    title = []\n",
    "    description = None\n",
    "    options = Options()\n",
    "    options.headless = False  # Set to True for headless mode\n",
    "    driver = webdriver.Chrome(options=options)\n",
    "\n",
    "    #Navigate to the webpage\n",
    "    driver.get(url)\n",
    "\n",
    "    #Allow time for dynamic content to load (you may need to use WebDriverWait for more robust waiting)\n",
    "    time.sleep(3)\n",
    "\n",
    "    for element in driver.find_elements(By.CLASS_NAME, \"item-meta-data__item-title\"):\n",
    "        #print(element.text)\n",
    "        title = element.text\n",
    "        #print(title)\n",
    "    #for element in driver.find_elements(By.CLASS_NAME, \"author-footnote-below-abstract\"):get.text() #retrieves all footnotes\n",
    "    for element in driver.find_elements(By.ID, \"FN1\"): #retrieves just footnote 1\n",
    "        #print(element.text)\n",
    "        footnote = element.text\n",
    "        #print(footnote)\n",
    "        #print(type(footnote))\n",
    "        \n",
    "    for element in driver.find_elements(By.PARTIAL_LINK_TEXT, \"doi.org\"):\n",
    "        doi = element.text\n",
    "        print(doi)\n",
    "    \n",
    "    for element in driver.find_elements(By.CSS_SELECTOR, \"div.tl-main-part.title\"): #finds section headers\n",
    "        #print(element.text)\n",
    "        counter += 1\n",
    "        description = element.text\n",
    "        print(description)\n",
    "        if \"Description of\" in description: \n",
    "            print('found', description)  \n",
    "            #snumber = 's' + str(counter - 4) + '/p[3]'\n",
    "            snumber = 's' + str(counter - 4)\n",
    "            print('snumber is', snumber)\n",
    "            for element in driver.find_elements(By.ID, snumber):\n",
    "                description = element.text\n",
    "                #print(description)\n",
    "        \n",
    "    #Close the browser window\n",
    "    driver.quit()\n",
    "    return title, footnote, description, doi\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "593675fc",
   "metadata": {},
   "source": [
    "### Beautifulsoup webscraper - returns abstract, can be turned off following additional testing. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5b0eff1c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def scrape_with_beautifulsoup(url):\n",
    "    global abstract_text\n",
    "    abstract_text = None\n",
    "    response = requests.get(url)\n",
    "    #soup = BeautifulSoup(response.content, \"html.parser\")\n",
    "    soup = BeautifulSoup(response.text, \"lxml\")\n",
    "    #print(soup.prettify())\n",
    "    abstract = soup.find(class_= \"articleabstract\")\n",
    "    if abstract is not None:\n",
    "    #print(abstract.get_text())\n",
    "        abstract_text = abstract.get_text()\n",
    "        return abstract_text\n",
    "    else:\n",
    "        return\n",
    "    "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.20"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
